/work2/09808/mp46753/frontera/ML_Project/VQGraph/vq.py:283: FutureWarning: `torch.cuda.amp.autocast(args...)` is deprecated. Please use `torch.amp.autocast('cuda', args...)` instead.
  @autocast(enabled=False)
/work2/09808/mp46753/frontera/ML_Project/VQGraph/vq.py:418: FutureWarning: `torch.cuda.amp.autocast(args...)` is deprecated. Please use `torch.amp.autocast('cuda', args...)` instead.
  @autocast(enabled=False)
/work2/09808/mp46753/frontera/ML_Project/miniconda3/envs/txgnn_env/lib/python3.8/site-packages/dgl/dataloading/dataloader.py:1144: DGLWarning: Dataloader CPU affinity opt is not enabled, consider switching it on (see enable_cpu_affinity() or CPU best practices for DGL [https://docs.dgl.ai/tutorials/cpu/cpu_best_practises.html])
  dgl_warning(
/work2/09808/mp46753/frontera/ML_Project/miniconda3/envs/txgnn_env/lib/python3.8/site-packages/dgl/heterograph.py:92: DGLWarning: Recommend creating graphs by `dgl.graph(data)` instead of `dgl.DGLGraph(data)`.
  dgl_warning(
/work2/09808/mp46753/frontera/ML_Project/miniconda3/envs/txgnn_env/lib/python3.8/site-packages/torch/_dynamo/variables/functions.py:663: UserWarning: Graph break due to unsupported builtin torch._C._to_dlpack. This function is either a Python builtin (e.g. _warnings.warn) or a third-party C/C++ Python extension (perhaps created with pybind). If it is a Python builtin, please file an issue on GitHub so the PyTorch team can add support for it and see the next case for a workaround. If it is a third-party C/C++ Python extension, please either wrap it into a PyTorch-understood custom operator (see https://pytorch.org/tutorials/advanced/custom_ops_landing_page.html for more details) or, if it is traceable, use torch.compiler.allow_in_graph.
  torch._dynamo.utils.warn_once(msg)
V0421 03:23:09.802685 47049167324416 torch/_dynamo/guards.py:2611] [4/1] [__recompiles] Recompiling function create_unitgraph_from_coo in /work2/09808/mp46753/frontera/ML_Project/miniconda3/envs/txgnn_env/lib/python3.8/site-packages/dgl/heterograph_index.py:1254
V0421 03:23:09.802685 47049167324416 torch/_dynamo/guards.py:2611] [4/1] [__recompiles]     triggered by the following guard failure(s):
V0421 03:23:09.802685 47049167324416 torch/_dynamo/guards.py:2611] [4/1] [__recompiles]     - tensor 'L['row']' dispatch key set mismatch. expected DispatchKeySet(CPU, BackendSelect, ADInplaceOrView, AutogradCPU), actual DispatchKeySet(CUDA, BackendSelect, ADInplaceOrView, AutogradCUDA)
V0421 03:23:09.822026 47049167324416 torch/_dynamo/guards.py:2611] [5/1] [__recompiles] Recompiling function to_dgl_nd in /work2/09808/mp46753/frontera/ML_Project/miniconda3/envs/txgnn_env/lib/python3.8/site-packages/dgl/backend/__init__.py:141
V0421 03:23:09.822026 47049167324416 torch/_dynamo/guards.py:2611] [5/1] [__recompiles]     triggered by the following guard failure(s):
V0421 03:23:09.822026 47049167324416 torch/_dynamo/guards.py:2611] [5/1] [__recompiles]     - tensor 'L['data']' dispatch key set mismatch. expected DispatchKeySet(CPU, BackendSelect, ADInplaceOrView, AutogradCPU), actual DispatchKeySet(CUDA, BackendSelect, ADInplaceOrView, AutogradCUDA)
V0421 03:23:09.831904 47049167324416 torch/_dynamo/guards.py:2611] [6/1] [__recompiles] Recompiling function zerocopy_to_dgl_ndarray in /work2/09808/mp46753/frontera/ML_Project/miniconda3/envs/txgnn_env/lib/python3.8/site-packages/dgl/backend/pytorch/tensor.py:432
V0421 03:23:09.831904 47049167324416 torch/_dynamo/guards.py:2611] [6/1] [__recompiles]     triggered by the following guard failure(s):
V0421 03:23:09.831904 47049167324416 torch/_dynamo/guards.py:2611] [6/1] [__recompiles]     - tensor 'L['data']' dispatch key set mismatch. expected DispatchKeySet(CPU, BackendSelect, ADInplaceOrView, AutogradCPU), actual DispatchKeySet(CUDA, BackendSelect, ADInplaceOrView, AutogradCUDA)
V0421 03:23:09.867164 47049167324416 torch/_dynamo/guards.py:2611] [8/1] [__recompiles] Recompiling function torch_dynamo_resume_in_create_unitgraph_from_coo_at_1296 in /work2/09808/mp46753/frontera/ML_Project/miniconda3/envs/txgnn_env/lib/python3.8/site-packages/dgl/heterograph_index.py:1296
V0421 03:23:09.867164 47049167324416 torch/_dynamo/guards.py:2611] [8/1] [__recompiles]     triggered by the following guard failure(s):
V0421 03:23:09.867164 47049167324416 torch/_dynamo/guards.py:2611] [8/1] [__recompiles]     - tensor 'L['col']' dispatch key set mismatch. expected DispatchKeySet(CPU, BackendSelect, ADInplaceOrView, AutogradCPU), actual DispatchKeySet(CUDA, BackendSelect, ADInplaceOrView, AutogradCUDA)
W0421 03:23:17.421154 47049167324416 torch/_dynamo/variables/tensor.py:715] [30/0] Graph break from `Tensor.item()`, consider setting:
W0421 03:23:17.421154 47049167324416 torch/_dynamo/variables/tensor.py:715] [30/0]     torch._dynamo.config.capture_scalar_outputs = True
W0421 03:23:17.421154 47049167324416 torch/_dynamo/variables/tensor.py:715] [30/0] or:
W0421 03:23:17.421154 47049167324416 torch/_dynamo/variables/tensor.py:715] [30/0]     env TORCHDYNAMO_CAPTURE_SCALAR_OUTPUTS=1
W0421 03:23:17.421154 47049167324416 torch/_dynamo/variables/tensor.py:715] [30/0] to include these operations in the captured graph.
W0421 03:23:17.421154 47049167324416 torch/_dynamo/variables/tensor.py:715] [30/0] 
V0421 03:23:20.377209 47049167324416 torch/_dynamo/guards.py:2611] [4/2] [__recompiles] Recompiling function create_unitgraph_from_coo in /work2/09808/mp46753/frontera/ML_Project/miniconda3/envs/txgnn_env/lib/python3.8/site-packages/dgl/heterograph_index.py:1254
V0421 03:23:20.377209 47049167324416 torch/_dynamo/guards.py:2611] [4/2] [__recompiles]     triggered by the following guard failure(s):
V0421 03:23:20.377209 47049167324416 torch/_dynamo/guards.py:2611] [4/2] [__recompiles]     - tensor 'L['row']' size mismatch at index 0. expected 0, actual 4952
V0421 03:23:20.377209 47049167324416 torch/_dynamo/guards.py:2611] [4/2] [__recompiles]     - tensor 'L['row']' dispatch key set mismatch. expected DispatchKeySet(CPU, BackendSelect, ADInplaceOrView, AutogradCPU), actual DispatchKeySet(CUDA, BackendSelect, ADInplaceOrView, AutogradCUDA)
V0421 03:23:20.405935 47049167324416 torch/_dynamo/guards.py:2611] [5/2] [__recompiles] Recompiling function to_dgl_nd in /work2/09808/mp46753/frontera/ML_Project/miniconda3/envs/txgnn_env/lib/python3.8/site-packages/dgl/backend/__init__.py:141
V0421 03:23:20.405935 47049167324416 torch/_dynamo/guards.py:2611] [5/2] [__recompiles]     triggered by the following guard failure(s):
V0421 03:23:20.405935 47049167324416 torch/_dynamo/guards.py:2611] [5/2] [__recompiles]     - tensor 'L['data']' size mismatch at index 0. expected 0, actual 4952
V0421 03:23:20.405935 47049167324416 torch/_dynamo/guards.py:2611] [5/2] [__recompiles]     - tensor 'L['data']' dispatch key set mismatch. expected DispatchKeySet(CPU, BackendSelect, ADInplaceOrView, AutogradCPU), actual DispatchKeySet(CUDA, BackendSelect, ADInplaceOrView, AutogradCUDA)
V0421 03:23:20.418446 47049167324416 torch/_dynamo/guards.py:2611] [6/2] [__recompiles] Recompiling function zerocopy_to_dgl_ndarray in /work2/09808/mp46753/frontera/ML_Project/miniconda3/envs/txgnn_env/lib/python3.8/site-packages/dgl/backend/pytorch/tensor.py:432
V0421 03:23:20.418446 47049167324416 torch/_dynamo/guards.py:2611] [6/2] [__recompiles]     triggered by the following guard failure(s):
V0421 03:23:20.418446 47049167324416 torch/_dynamo/guards.py:2611] [6/2] [__recompiles]     - tensor 'L['data']' size mismatch at index 0. expected 0, actual 4952
V0421 03:23:20.418446 47049167324416 torch/_dynamo/guards.py:2611] [6/2] [__recompiles]     - tensor 'L['data']' dispatch key set mismatch. expected DispatchKeySet(CPU, BackendSelect, ADInplaceOrView, AutogradCPU), actual DispatchKeySet(CUDA, BackendSelect, ADInplaceOrView, AutogradCUDA)
V0421 03:23:20.456933 47049167324416 torch/_dynamo/guards.py:2611] [8/2] [__recompiles] Recompiling function torch_dynamo_resume_in_create_unitgraph_from_coo_at_1296 in /work2/09808/mp46753/frontera/ML_Project/miniconda3/envs/txgnn_env/lib/python3.8/site-packages/dgl/heterograph_index.py:1296
V0421 03:23:20.456933 47049167324416 torch/_dynamo/guards.py:2611] [8/2] [__recompiles]     triggered by the following guard failure(s):
V0421 03:23:20.456933 47049167324416 torch/_dynamo/guards.py:2611] [8/2] [__recompiles]     - tensor 'L['col']' size mismatch at index 0. expected 0, actual 4952
V0421 03:23:20.456933 47049167324416 torch/_dynamo/guards.py:2611] [8/2] [__recompiles]     - tensor 'L['col']' dispatch key set mismatch. expected DispatchKeySet(CPU, BackendSelect, ADInplaceOrView, AutogradCPU), actual DispatchKeySet(CUDA, BackendSelect, ADInplaceOrView, AutogradCUDA)
V0421 03:23:20.473278 47049167324416 torch/_dynamo/guards.py:2611] [32/1] [__recompiles] Recompiling function torch_dynamo_resume_in_add_edges_at_574 in /work2/09808/mp46753/frontera/ML_Project/miniconda3/envs/txgnn_env/lib/python3.8/site-packages/dgl/heterograph.py:574
V0421 03:23:20.473278 47049167324416 torch/_dynamo/guards.py:2611] [32/1] [__recompiles]     triggered by the following guard failure(s):
V0421 03:23:20.473278 47049167324416 torch/_dynamo/guards.py:2611] [32/1] [__recompiles]     - L['___stack0'] == 3523                                      
V0421 03:23:20.696503 47049167324416 torch/_dynamo/guards.py:2611] [34/1] [__recompiles] Recompiling function cat in /work2/09808/mp46753/frontera/ML_Project/miniconda3/envs/txgnn_env/lib/python3.8/site-packages/dgl/backend/pytorch/tensor.py:222
V0421 03:23:20.696503 47049167324416 torch/_dynamo/guards.py:2611] [34/1] [__recompiles]     triggered by the following guard failure(s):
V0421 03:23:20.696503 47049167324416 torch/_dynamo/guards.py:2611] [34/1] [__recompiles]     - tensor 'L['seq'][0]' size mismatch at index 0. expected 0, actual 4952
V0421 03:23:21.441055 47049167324416 torch/_dynamo/guards.py:2611] [20/1] [__recompiles] Recompiling function add_edges in /work2/09808/mp46753/frontera/ML_Project/miniconda3/envs/txgnn_env/lib/python3.8/site-packages/dgl/heterograph.py:435
V0421 03:23:21.441055 47049167324416 torch/_dynamo/guards.py:2611] [20/1] [__recompiles]     triggered by the following guard failure(s):
V0421 03:23:21.441055 47049167324416 torch/_dynamo/guards.py:2611] [20/1] [__recompiles]     - tensor 'L['u']' size mismatch at index 0. expected 4952, actual 984
V0421 03:23:21.463341 47049167324416 torch/_dynamo/guards.py:2611] [26/1] [__recompiles] Recompiling function torch_dynamo_resume_in_add_edges_at_541 in /work2/09808/mp46753/frontera/ML_Project/miniconda3/envs/txgnn_env/lib/python3.8/site-packages/dgl/heterograph.py:541
V0421 03:23:21.463341 47049167324416 torch/_dynamo/guards.py:2611] [26/1] [__recompiles]     triggered by the following guard failure(s):
V0421 03:23:21.463341 47049167324416 torch/_dynamo/guards.py:2611] [26/1] [__recompiles]     - tensor 'L['v']' size mismatch at index 0. expected 4952, actual 984
V0421 03:23:21.490675 47049167324416 torch/_dynamo/guards.py:2611] [27/1] [__recompiles] Recompiling function torch_dynamo_resume_in_add_edges_at_542 in /work2/09808/mp46753/frontera/ML_Project/miniconda3/envs/txgnn_env/lib/python3.8/site-packages/dgl/heterograph.py:542
V0421 03:23:21.490675 47049167324416 torch/_dynamo/guards.py:2611] [27/1] [__recompiles]     triggered by the following guard failure(s):
V0421 03:23:21.490675 47049167324416 torch/_dynamo/guards.py:2611] [27/1] [__recompiles]     - tensor 'L['___stack0']' size mismatch at index 0. expected 4952, actual 984
V0421 03:23:21.507003 47049167324416 torch/_dynamo/guards.py:2611] [28/1] [__recompiles] Recompiling function torch_dynamo_resume_in_add_edges_at_545 in /work2/09808/mp46753/frontera/ML_Project/miniconda3/envs/txgnn_env/lib/python3.8/site-packages/dgl/heterograph.py:545
V0421 03:23:21.507003 47049167324416 torch/_dynamo/guards.py:2611] [28/1] [__recompiles]     triggered by the following guard failure(s):
V0421 03:23:21.507003 47049167324416 torch/_dynamo/guards.py:2611] [28/1] [__recompiles]     - tensor 'L['u']' size mismatch at index 0. expected 4952, actual 984
V0421 03:23:21.573230 47049167324416 torch/_dynamo/guards.py:2611] [30/1] [__recompiles] Recompiling function torch_dynamo_resume_in_add_edges_at_573 in /work2/09808/mp46753/frontera/ML_Project/miniconda3/envs/txgnn_env/lib/python3.8/site-packages/dgl/heterograph.py:573
V0421 03:23:21.573230 47049167324416 torch/_dynamo/guards.py:2611] [30/1] [__recompiles]     triggered by the following guard failure(s):
V0421 03:23:21.573230 47049167324416 torch/_dynamo/guards.py:2611] [30/1] [__recompiles]     - tensor 'L['u']' size mismatch at index 0. expected 4952, actual 984
V0421 03:23:21.828844 47049167324416 torch/_dynamo/guards.py:2611] [32/2] [__recompiles] Recompiling function torch_dynamo_resume_in_add_edges_at_574 in /work2/09808/mp46753/frontera/ML_Project/miniconda3/envs/txgnn_env/lib/python3.8/site-packages/dgl/heterograph.py:574
V0421 03:23:21.828844 47049167324416 torch/_dynamo/guards.py:2611] [32/2] [__recompiles]     triggered by the following guard failure(s):
V0421 03:23:21.828844 47049167324416 torch/_dynamo/guards.py:2611] [32/2] [__recompiles]     - tensor 'L['v']' size mismatch at index 0. expected 4952, actual 984
V0421 03:23:21.828844 47049167324416 torch/_dynamo/guards.py:2611] [32/2] [__recompiles]     - L['___stack0'] == 3523                                      
V0421 03:23:22.080959 47049167324416 torch/_dynamo/guards.py:2611] [34/2] [__recompiles] Recompiling function cat in /work2/09808/mp46753/frontera/ML_Project/miniconda3/envs/txgnn_env/lib/python3.8/site-packages/dgl/backend/pytorch/tensor.py:222
V0421 03:23:22.080959 47049167324416 torch/_dynamo/guards.py:2611] [34/2] [__recompiles]     triggered by the following guard failure(s):
V0421 03:23:22.080959 47049167324416 torch/_dynamo/guards.py:2611] [34/2] [__recompiles]     - tensor 'L['seq'][1]' size mismatch at index 0. expected 4952, actual 984
V0421 03:23:22.080959 47049167324416 torch/_dynamo/guards.py:2611] [34/2] [__recompiles]     - tensor 'L['seq'][0]' size mismatch at index 0. expected 0, actual 9904
/work2/09808/mp46753/frontera/ML_Project/miniconda3/envs/txgnn_env/lib/python3.8/site-packages/torch/_dynamo/variables/functions.py:663: UserWarning: Graph break due to unsupported builtin posix.stat. This function is either a Python builtin (e.g. _warnings.warn) or a third-party C/C++ Python extension (perhaps created with pybind). If it is a Python builtin, please file an issue on GitHub so the PyTorch team can add support for it and see the next case for a workaround. If it is a third-party C/C++ Python extension, please either wrap it into a PyTorch-understood custom operator (see https://pytorch.org/tutorials/advanced/custom_ops_landing_page.html for more details) or, if it is traceable, use torch.compiler.allow_in_graph.
  torch._dynamo.utils.warn_once(msg)
V0421 03:23:26.264759 47049167324416 torch/_dynamo/guards.py:2611] [6/3] [__recompiles] Recompiling function zerocopy_to_dgl_ndarray in /work2/09808/mp46753/frontera/ML_Project/miniconda3/envs/txgnn_env/lib/python3.8/site-packages/dgl/backend/pytorch/tensor.py:432
V0421 03:23:26.264759 47049167324416 torch/_dynamo/guards.py:2611] [6/3] [__recompiles]     triggered by the following guard failure(s):
V0421 03:23:26.264759 47049167324416 torch/_dynamo/guards.py:2611] [6/3] [__recompiles]     - GLOBAL_STATE changed: grad_mode 
V0421 03:23:26.264759 47049167324416 torch/_dynamo/guards.py:2611] [6/3] [__recompiles]     - GLOBAL_STATE changed: grad_mode 
V0421 03:23:26.264759 47049167324416 torch/_dynamo/guards.py:2611] [6/3] [__recompiles]     - GLOBAL_STATE changed: grad_mode 
V0421 03:23:26.311604 47049167324416 torch/_dynamo/guards.py:2611] [7/1] [__recompiles] Recompiling function torch_dynamo_resume_in_zerocopy_to_dgl_ndarray_at_435 in /work2/09808/mp46753/frontera/ML_Project/miniconda3/envs/txgnn_env/lib/python3.8/site-packages/dgl/backend/pytorch/tensor.py:435
V0421 03:23:26.311604 47049167324416 torch/_dynamo/guards.py:2611] [7/1] [__recompiles]     triggered by the following guard failure(s):
V0421 03:23:26.311604 47049167324416 torch/_dynamo/guards.py:2611] [7/1] [__recompiles]     - GLOBAL_STATE changed: grad_mode 
/work2/09808/mp46753/frontera/ML_Project/miniconda3/envs/txgnn_env/lib/python3.8/site-packages/torch/_dynamo/variables/functions.py:663: UserWarning: Graph break due to unsupported builtin time.perf_counter. This function is either a Python builtin (e.g. _warnings.warn) or a third-party C/C++ Python extension (perhaps created with pybind). If it is a Python builtin, please file an issue on GitHub so the PyTorch team can add support for it and see the next case for a workaround. If it is a third-party C/C++ Python extension, please either wrap it into a PyTorch-understood custom operator (see https://pytorch.org/tutorials/advanced/custom_ops_landing_page.html for more details) or, if it is traceable, use torch.compiler.allow_in_graph.
  torch._dynamo.utils.warn_once(msg)
V0421 03:23:26.556797 47049167324416 torch/_dynamo/guards.py:2611] [86/1] [__recompiles] Recompiling function torch_dynamo_resume_in_hook_at_12 in /work2/09808/mp46753/frontera/ML_Project/VQGraph/benchmark_utils.py:12
V0421 03:23:26.556797 47049167324416 torch/_dynamo/guards.py:2611] [86/1] [__recompiles]     triggered by the following guard failure(s):
V0421 03:23:26.556797 47049167324416 torch/_dynamo/guards.py:2611] [86/1] [__recompiles]     - L['___stack0'] == 927248.028905784                          
V0421 03:23:26.649389 47049167324416 torch/_dynamo/guards.py:2611] [86/2] [__recompiles] Recompiling function torch_dynamo_resume_in_hook_at_12 in /work2/09808/mp46753/frontera/ML_Project/VQGraph/benchmark_utils.py:12
V0421 03:23:26.649389 47049167324416 torch/_dynamo/guards.py:2611] [86/2] [__recompiles]     triggered by the following guard failure(s):
V0421 03:23:26.649389 47049167324416 torch/_dynamo/guards.py:2611] [86/2] [__recompiles]     - L['___stack0'] == 927248.041685607                          
V0421 03:23:26.649389 47049167324416 torch/_dynamo/guards.py:2611] [86/2] [__recompiles]     - L['___stack0'] == 927248.028905784                          
V0421 03:23:29.793427 47049167324416 torch/_dynamo/guards.py:2611] [86/3] [__recompiles] Recompiling function torch_dynamo_resume_in_hook_at_12 in /work2/09808/mp46753/frontera/ML_Project/VQGraph/benchmark_utils.py:12
V0421 03:23:29.793427 47049167324416 torch/_dynamo/guards.py:2611] [86/3] [__recompiles]     triggered by the following guard failure(s):
V0421 03:23:29.793427 47049167324416 torch/_dynamo/guards.py:2611] [86/3] [__recompiles]     - L['___stack0'] == 927248.134274053                          
V0421 03:23:29.793427 47049167324416 torch/_dynamo/guards.py:2611] [86/3] [__recompiles]     - L['___stack0'] == 927248.041685607                          
V0421 03:23:29.793427 47049167324416 torch/_dynamo/guards.py:2611] [86/3] [__recompiles]     - L['___stack0'] == 927248.028905784                          
V0421 03:23:31.347472 47049167324416 torch/_dynamo/guards.py:2611] [86/4] [__recompiles] Recompiling function torch_dynamo_resume_in_hook_at_12 in /work2/09808/mp46753/frontera/ML_Project/VQGraph/benchmark_utils.py:12
V0421 03:23:31.347472 47049167324416 torch/_dynamo/guards.py:2611] [86/4] [__recompiles]     triggered by the following guard failure(s):
V0421 03:23:31.347472 47049167324416 torch/_dynamo/guards.py:2611] [86/4] [__recompiles]     - L['___stack0'] == 927251.278203949                          
V0421 03:23:31.347472 47049167324416 torch/_dynamo/guards.py:2611] [86/4] [__recompiles]     - L['___stack0'] == 927248.134274053                          
V0421 03:23:31.347472 47049167324416 torch/_dynamo/guards.py:2611] [86/4] [__recompiles]     - L['___stack0'] == 927248.041685607                          
V0421 03:23:31.347472 47049167324416 torch/_dynamo/guards.py:2611] [86/4] [__recompiles]     - L['___stack0'] == 927248.028905784                          
V0421 03:23:31.688541 47049167324416 torch/_dynamo/guards.py:2611] [86/5] [__recompiles] Recompiling function torch_dynamo_resume_in_hook_at_12 in /work2/09808/mp46753/frontera/ML_Project/VQGraph/benchmark_utils.py:12
V0421 03:23:31.688541 47049167324416 torch/_dynamo/guards.py:2611] [86/5] [__recompiles]     triggered by the following guard failure(s):
V0421 03:23:31.688541 47049167324416 torch/_dynamo/guards.py:2611] [86/5] [__recompiles]     - L['___stack0'] == 927252.831826534                          
V0421 03:23:31.688541 47049167324416 torch/_dynamo/guards.py:2611] [86/5] [__recompiles]     - L['___stack0'] == 927251.278203949                          
V0421 03:23:31.688541 47049167324416 torch/_dynamo/guards.py:2611] [86/5] [__recompiles]     - L['___stack0'] == 927248.134274053                          
V0421 03:23:31.688541 47049167324416 torch/_dynamo/guards.py:2611] [86/5] [__recompiles]     - L['___stack0'] == 927248.041685607                          
V0421 03:23:31.688541 47049167324416 torch/_dynamo/guards.py:2611] [86/5] [__recompiles]     - L['___stack0'] == 927248.028905784                          
V0421 03:23:31.695362 47049167324416 torch/_dynamo/guards.py:2611] [86/6] [__recompiles] Recompiling function torch_dynamo_resume_in_hook_at_12 in /work2/09808/mp46753/frontera/ML_Project/VQGraph/benchmark_utils.py:12
V0421 03:23:31.695362 47049167324416 torch/_dynamo/guards.py:2611] [86/6] [__recompiles]     triggered by the following guard failure(s):
V0421 03:23:31.695362 47049167324416 torch/_dynamo/guards.py:2611] [86/6] [__recompiles]     - L['___stack0'] == 927253.173220772                          
V0421 03:23:31.695362 47049167324416 torch/_dynamo/guards.py:2611] [86/6] [__recompiles]     - L['___stack0'] == 927252.831826534                          
V0421 03:23:31.695362 47049167324416 torch/_dynamo/guards.py:2611] [86/6] [__recompiles]     - L['___stack0'] == 927251.278203949                          
V0421 03:23:31.695362 47049167324416 torch/_dynamo/guards.py:2611] [86/6] [__recompiles]     - L['___stack0'] == 927248.134274053                          
V0421 03:23:31.695362 47049167324416 torch/_dynamo/guards.py:2611] [86/6] [__recompiles]     - L['___stack0'] == 927248.041685607                          
V0421 03:23:31.695362 47049167324416 torch/_dynamo/guards.py:2611] [86/6] [__recompiles]     - L['___stack0'] == 927248.028905784                          
V0421 03:23:31.755630 47049167324416 torch/_dynamo/guards.py:2611] [57/1] [__recompiles] Recompiling function __setitem__ in /work2/09808/mp46753/frontera/ML_Project/miniconda3/envs/txgnn_env/lib/python3.8/site-packages/dgl/view.py:82
V0421 03:23:31.755630 47049167324416 torch/_dynamo/guards.py:2611] [57/1] [__recompiles]     triggered by the following guard failure(s):
V0421 03:23:31.755630 47049167324416 torch/_dynamo/guards.py:2611] [57/1] [__recompiles]     - tensor 'L['val']' requires_grad mismatch. expected requires_grad=0
V0421 03:23:31.777861 47049167324416 torch/_dynamo/guards.py:2611] [61/1] [__recompiles] Recompiling function __setitem__ in /work2/09808/mp46753/frontera/ML_Project/miniconda3/envs/txgnn_env/lib/python3.8/site-packages/dgl/frame.py:690
V0421 03:23:31.777861 47049167324416 torch/_dynamo/guards.py:2611] [61/1] [__recompiles]     triggered by the following guard failure(s):
V0421 03:23:31.777861 47049167324416 torch/_dynamo/guards.py:2611] [61/1] [__recompiles]     - tensor 'L['data']' requires_grad mismatch. expected requires_grad=0
V0421 03:23:31.797277 47049167324416 torch/_dynamo/guards.py:2611] [62/1] [__recompiles] Recompiling function update_column in /work2/09808/mp46753/frontera/ML_Project/miniconda3/envs/txgnn_env/lib/python3.8/site-packages/dgl/frame.py:772
V0421 03:23:31.797277 47049167324416 torch/_dynamo/guards.py:2611] [62/1] [__recompiles]     triggered by the following guard failure(s):
V0421 03:23:31.797277 47049167324416 torch/_dynamo/guards.py:2611] [62/1] [__recompiles]     - tensor 'L['data']' requires_grad mismatch. expected requires_grad=0
V0421 03:23:31.817058 47049167324416 torch/_dynamo/guards.py:2611] [63/1] [__recompiles] Recompiling function create in /work2/09808/mp46753/frontera/ML_Project/miniconda3/envs/txgnn_env/lib/python3.8/site-packages/dgl/frame.py:472
V0421 03:23:31.817058 47049167324416 torch/_dynamo/guards.py:2611] [63/1] [__recompiles]     triggered by the following guard failure(s):
V0421 03:23:31.817058 47049167324416 torch/_dynamo/guards.py:2611] [63/1] [__recompiles]     - tensor 'L['data']' requires_grad mismatch. expected requires_grad=0
V0421 03:23:31.835652 47049167324416 torch/_dynamo/guards.py:2611] [64/1] [__recompiles] Recompiling function __init__ in /work2/09808/mp46753/frontera/ML_Project/miniconda3/envs/txgnn_env/lib/python3.8/site-packages/dgl/frame.py:210
V0421 03:23:31.835652 47049167324416 torch/_dynamo/guards.py:2611] [64/1] [__recompiles]     triggered by the following guard failure(s):
V0421 03:23:31.835652 47049167324416 torch/_dynamo/guards.py:2611] [64/1] [__recompiles]     - tensor 'L['storage']' requires_grad mismatch. expected requires_grad=0
V0421 03:23:31.855242 47049167324416 torch/_dynamo/guards.py:2611] [65/1] [__recompiles] Recompiling function infer_scheme in /work2/09808/mp46753/frontera/ML_Project/miniconda3/envs/txgnn_env/lib/python3.8/site-packages/dgl/frame.py:151
V0421 03:23:31.855242 47049167324416 torch/_dynamo/guards.py:2611] [65/1] [__recompiles]     triggered by the following guard failure(s):
V0421 03:23:31.855242 47049167324416 torch/_dynamo/guards.py:2611] [65/1] [__recompiles]     - tensor 'L['tensor']' requires_grad mismatch. expected requires_grad=0
V0421 03:23:31.900860 47049167324416 torch/_dynamo/guards.py:2611] [68/1] [__recompiles] Recompiling function func in /work2/09808/mp46753/frontera/ML_Project/miniconda3/envs/txgnn_env/lib/python3.8/site-packages/dgl/ops/spmm.py:213
V0421 03:23:31.900860 47049167324416 torch/_dynamo/guards.py:2611] [68/1] [__recompiles]     triggered by the following guard failure(s):
V0421 03:23:31.900860 47049167324416 torch/_dynamo/guards.py:2611] [68/1] [__recompiles]     - tensor 'L['x']' requires_grad mismatch. expected requires_grad=0
V0421 03:23:31.914469 47049167324416 torch/_dynamo/guards.py:2611] [70/1] [__recompiles] Recompiling function torch_dynamo_resume_in_gspmm_at_75 in /work2/09808/mp46753/frontera/ML_Project/miniconda3/envs/txgnn_env/lib/python3.8/site-packages/dgl/ops/spmm.py:75
V0421 03:23:31.914469 47049167324416 torch/_dynamo/guards.py:2611] [70/1] [__recompiles]     triggered by the following guard failure(s):
V0421 03:23:31.914469 47049167324416 torch/_dynamo/guards.py:2611] [70/1] [__recompiles]     - tensor 'L['lhs_data']' requires_grad mismatch. expected requires_grad=0
V0421 03:23:31.932297 47049167324416 torch/_dynamo/guards.py:2611] [74/1] [__recompiles] Recompiling function forward in /work2/09808/mp46753/frontera/ML_Project/miniconda3/envs/txgnn_env/lib/python3.8/site-packages/dgl/backend/pytorch/sparse.py:163
V0421 03:23:31.932297 47049167324416 torch/_dynamo/guards.py:2611] [74/1] [__recompiles]     triggered by the following guard failure(s):
V0421 03:23:31.932297 47049167324416 torch/_dynamo/guards.py:2611] [74/1] [__recompiles]     - tensor 'L['X']' requires_grad mismatch. expected requires_grad=0
V0421 03:23:31.948642 47049167324416 torch/_dynamo/guards.py:2611] [77/1] [__recompiles] Recompiling function zeros in /work2/09808/mp46753/frontera/ML_Project/miniconda3/envs/txgnn_env/lib/python3.8/site-packages/dgl/backend/pytorch/tensor.py:287
V0421 03:23:31.948642 47049167324416 torch/_dynamo/guards.py:2611] [77/1] [__recompiles]     triggered by the following guard failure(s):
V0421 03:23:31.948642 47049167324416 torch/_dynamo/guards.py:2611] [77/1] [__recompiles]     - L['shape'][1] == 767                                        
V0421 03:23:32.233387 47049167324416 torch/_dynamo/guards.py:2611] [6/4] [__recompiles] Recompiling function zerocopy_to_dgl_ndarray in /work2/09808/mp46753/frontera/ML_Project/miniconda3/envs/txgnn_env/lib/python3.8/site-packages/dgl/backend/pytorch/tensor.py:432
V0421 03:23:32.233387 47049167324416 torch/_dynamo/guards.py:2611] [6/4] [__recompiles]     triggered by the following guard failure(s):
V0421 03:23:32.233387 47049167324416 torch/_dynamo/guards.py:2611] [6/4] [__recompiles]     - GLOBAL_STATE changed: grad_mode 
V0421 03:23:32.233387 47049167324416 torch/_dynamo/guards.py:2611] [6/4] [__recompiles]     - tensor 'L['data']' requires_grad mismatch. expected requires_grad=0
V0421 03:23:32.233387 47049167324416 torch/_dynamo/guards.py:2611] [6/4] [__recompiles]     - GLOBAL_STATE changed: grad_mode 
V0421 03:23:32.233387 47049167324416 torch/_dynamo/guards.py:2611] [6/4] [__recompiles]     - GLOBAL_STATE changed: grad_mode 
V0421 03:23:32.283449 47049167324416 torch/_dynamo/guards.py:2611] [79/1] [__recompiles] Recompiling function zerocopy_to_dgl_ndarray_for_write in /work2/09808/mp46753/frontera/ML_Project/miniconda3/envs/txgnn_env/lib/python3.8/site-packages/dgl/backend/pytorch/tensor.py:454
V0421 03:23:32.283449 47049167324416 torch/_dynamo/guards.py:2611] [79/1] [__recompiles]     triggered by the following guard failure(s):
V0421 03:23:32.283449 47049167324416 torch/_dynamo/guards.py:2611] [79/1] [__recompiles]     - tensor 'L['input']' stride mismatch at index 0. expected 767, actual 128
V0421 03:23:32.320960 47049167324416 torch/_dynamo/guards.py:2611] [80/1] [__recompiles] Recompiling function check_is_view in /work2/09808/mp46753/frontera/ML_Project/miniconda3/envs/txgnn_env/lib/python3.8/site-packages/dgl/backend/pytorch/tensor.py:441
V0421 03:23:32.320960 47049167324416 torch/_dynamo/guards.py:2611] [80/1] [__recompiles]     triggered by the following guard failure(s):
V0421 03:23:32.320960 47049167324416 torch/_dynamo/guards.py:2611] [80/1] [__recompiles]     - tensor 'L['input']' stride mismatch at index 0. expected 767, actual 128
V0421 03:23:32.331144 47049167324416 torch/_dynamo/guards.py:2611] [81/1] [__recompiles] Recompiling function torch_dynamo_resume_in_check_is_view_at_443 in /work2/09808/mp46753/frontera/ML_Project/miniconda3/envs/txgnn_env/lib/python3.8/site-packages/dgl/backend/pytorch/tensor.py:443
V0421 03:23:32.331144 47049167324416 torch/_dynamo/guards.py:2611] [81/1] [__recompiles]     triggered by the following guard failure(s):
V0421 03:23:32.331144 47049167324416 torch/_dynamo/guards.py:2611] [81/1] [__recompiles]     - tensor 'L['input']' stride mismatch at index 0. expected 767, actual 128
V0421 03:23:32.342735 47049167324416 torch/_dynamo/guards.py:2611] [82/1] [__recompiles] Recompiling function torch_dynamo_resume_in_zerocopy_to_dgl_ndarray_for_write_at_461 in /work2/09808/mp46753/frontera/ML_Project/miniconda3/envs/txgnn_env/lib/python3.8/site-packages/dgl/backend/pytorch/tensor.py:461
V0421 03:23:32.342735 47049167324416 torch/_dynamo/guards.py:2611] [82/1] [__recompiles]     triggered by the following guard failure(s):
V0421 03:23:32.342735 47049167324416 torch/_dynamo/guards.py:2611] [82/1] [__recompiles]     - tensor 'L['input']' stride mismatch at index 0. expected 767, actual 128
V0421 03:23:32.356418 47049167324416 torch/_dynamo/guards.py:2611] [83/1] [__recompiles] Recompiling function torch_dynamo_resume_in_forward_at_165 in /work2/09808/mp46753/frontera/ML_Project/miniconda3/envs/txgnn_env/lib/python3.8/site-packages/dgl/backend/pytorch/sparse.py:165
V0421 03:23:32.356418 47049167324416 torch/_dynamo/guards.py:2611] [83/1] [__recompiles]     triggered by the following guard failure(s):
V0421 03:23:32.356418 47049167324416 torch/_dynamo/guards.py:2611] [83/1] [__recompiles]     - tensor 'L['X']' requires_grad mismatch. expected requires_grad=0
V0421 03:23:32.396856 47049167324416 torch/_dynamo/guards.py:2611] [86/7] [__recompiles] Recompiling function torch_dynamo_resume_in_hook_at_12 in /work2/09808/mp46753/frontera/ML_Project/VQGraph/benchmark_utils.py:12
V0421 03:23:32.396856 47049167324416 torch/_dynamo/guards.py:2611] [86/7] [__recompiles]     triggered by the following guard failure(s):
V0421 03:23:32.396856 47049167324416 torch/_dynamo/guards.py:2611] [86/7] [__recompiles]     - L['___stack0'] == 927253.180205768                          
V0421 03:23:32.396856 47049167324416 torch/_dynamo/guards.py:2611] [86/7] [__recompiles]     - L['___stack0'] == 927253.173220772                          
V0421 03:23:32.396856 47049167324416 torch/_dynamo/guards.py:2611] [86/7] [__recompiles]     - L['___stack0'] == 927252.831826534                          
V0421 03:23:32.396856 47049167324416 torch/_dynamo/guards.py:2611] [86/7] [__recompiles]     - L['___stack0'] == 927251.278203949                          
V0421 03:23:32.396856 47049167324416 torch/_dynamo/guards.py:2611] [86/7] [__recompiles]     - L['___stack0'] == 927248.134274053                          
V0421 03:23:32.396856 47049167324416 torch/_dynamo/guards.py:2611] [86/7] [__recompiles]     - L['___stack0'] == 927248.041685607                          
V0421 03:23:32.396856 47049167324416 torch/_dynamo/guards.py:2611] [86/7] [__recompiles]     - L['___stack0'] == 927248.028905784                          
V0421 03:23:32.403257 47049167324416 torch/_dynamo/guards.py:2611] [86/8] [__recompiles] Recompiling function torch_dynamo_resume_in_hook_at_12 in /work2/09808/mp46753/frontera/ML_Project/VQGraph/benchmark_utils.py:12
V0421 03:23:32.403257 47049167324416 torch/_dynamo/guards.py:2611] [86/8] [__recompiles]     triggered by the following guard failure(s):
V0421 03:23:32.403257 47049167324416 torch/_dynamo/guards.py:2611] [86/8] [__recompiles]     - L['___stack0'] == 927253.881638935                          
V0421 03:23:32.403257 47049167324416 torch/_dynamo/guards.py:2611] [86/8] [__recompiles]     - L['___stack0'] == 927253.180205768                          
V0421 03:23:32.403257 47049167324416 torch/_dynamo/guards.py:2611] [86/8] [__recompiles]     - L['___stack0'] == 927253.173220772                          
V0421 03:23:32.403257 47049167324416 torch/_dynamo/guards.py:2611] [86/8] [__recompiles]     - L['___stack0'] == 927252.831826534                          
V0421 03:23:32.403257 47049167324416 torch/_dynamo/guards.py:2611] [86/8] [__recompiles]     - L['___stack0'] == 927251.278203949                          
V0421 03:23:32.403257 47049167324416 torch/_dynamo/guards.py:2611] [86/8] [__recompiles]     - L['___stack0'] == 927248.134274053                          
V0421 03:23:32.403257 47049167324416 torch/_dynamo/guards.py:2611] [86/8] [__recompiles]     - L['___stack0'] == 927248.041685607                          
V0421 03:23:32.403257 47049167324416 torch/_dynamo/guards.py:2611] [86/8] [__recompiles]     - L['___stack0'] == 927248.028905784                          
W0421 03:23:32.403394 47049167324416 torch/_dynamo/convert_frame.py:762] [86/8] torch._dynamo hit config.cache_size_limit (8)
W0421 03:23:32.403394 47049167324416 torch/_dynamo/convert_frame.py:762] [86/8]    function: 'torch_dynamo_resume_in_hook_at_12' (/work2/09808/mp46753/frontera/ML_Project/VQGraph/benchmark_utils.py:12)
W0421 03:23:32.403394 47049167324416 torch/_dynamo/convert_frame.py:762] [86/8]    last reason: L['___stack0'] == 927248.028905784                          
W0421 03:23:32.403394 47049167324416 torch/_dynamo/convert_frame.py:762] [86/8] To log all recompilation reasons, use TORCH_LOGS="recompiles".
W0421 03:23:32.403394 47049167324416 torch/_dynamo/convert_frame.py:762] [86/8] To diagnose recompilation issues, see https://pytorch.org/docs/main/torch.compiler_troubleshooting.html.
/work2/09808/mp46753/frontera/ML_Project/miniconda3/envs/txgnn_env/lib/python3.8/site-packages/dgl/dataloading/dataloader.py:1144: DGLWarning: Dataloader CPU affinity opt is not enabled, consider switching it on (see enable_cpu_affinity() or CPU best practices for DGL [https://docs.dgl.ai/tutorials/cpu/cpu_best_practises.html])
  dgl_warning(
/work2/09808/mp46753/frontera/ML_Project/miniconda3/envs/txgnn_env/lib/python3.8/site-packages/dgl/heterograph.py:92: DGLWarning: Recommend creating graphs by `dgl.graph(data)` instead of `dgl.DGLGraph(data)`.
  dgl_warning(
V0421 03:23:34.262500 47049167324416 torch/_dynamo/guards.py:2611] [0/1] [__recompiles] Recompiling function forward in /work2/09808/mp46753/frontera/ML_Project/VQGraph/models.py:449
V0421 03:23:34.262500 47049167324416 torch/_dynamo/guards.py:2611] [0/1] [__recompiles]     triggered by the following guard failure(s):
V0421 03:23:34.262500 47049167324416 torch/_dynamo/guards.py:2611] [0/1] [__recompiles]     - tensor 'L['feats']' size mismatch at index 0. expected 3524, actual 3433
V0421 03:23:34.301185 47049167324416 torch/_dynamo/guards.py:2611] [1/1] [__recompiles] Recompiling function forward in /work2/09808/mp46753/frontera/ML_Project/VQGraph/models.py:154
V0421 03:23:34.301185 47049167324416 torch/_dynamo/guards.py:2611] [1/1] [__recompiles]     triggered by the following guard failure(s):
V0421 03:23:34.301185 47049167324416 torch/_dynamo/guards.py:2611] [1/1] [__recompiles]     - tensor 'L['feats']' size mismatch at index 0. expected 3524, actual 3433
/work2/09808/mp46753/frontera/ML_Project/miniconda3/envs/txgnn_env/lib/python3.8/site-packages/dgl/heterograph.py:92: DGLWarning: Recommend creating graphs by `dgl.graph(data)` instead of `dgl.DGLGraph(data)`.
  dgl_warning(
V0421 03:23:34.319159 47049167324416 torch/_dynamo/guards.py:2611] [9/1] [__recompiles] Recompiling function torch_dynamo_resume_in_forward_at_157 in /work2/09808/mp46753/frontera/ML_Project/VQGraph/models.py:157
V0421 03:23:34.319159 47049167324416 torch/_dynamo/guards.py:2611] [9/1] [__recompiles]     triggered by the following guard failure(s):
V0421 03:23:34.319159 47049167324416 torch/_dynamo/guards.py:2611] [9/1] [__recompiles]     - tensor 'L['h']' size mismatch at index 0. expected 3524, actual 3433
V0421 03:23:34.335674 47049167324416 torch/_dynamo/guards.py:2611] [13/1] [__recompiles] Recompiling function torch_dynamo_resume_in_forward_at_157 in /work2/09808/mp46753/frontera/ML_Project/VQGraph/models.py:157
V0421 03:23:34.335674 47049167324416 torch/_dynamo/guards.py:2611] [13/1] [__recompiles]     triggered by the following guard failure(s):
V0421 03:23:34.335674 47049167324416 torch/_dynamo/guards.py:2611] [13/1] [__recompiles]     - tensor 'L['h']' size mismatch at index 0. expected 3524, actual 3433
V0421 03:23:34.383411 47049167324416 torch/_dynamo/guards.py:2611] [4/3] [__recompiles] Recompiling function create_unitgraph_from_coo in /work2/09808/mp46753/frontera/ML_Project/miniconda3/envs/txgnn_env/lib/python3.8/site-packages/dgl/heterograph_index.py:1254
V0421 03:23:34.383411 47049167324416 torch/_dynamo/guards.py:2611] [4/3] [__recompiles]     triggered by the following guard failure(s):
V0421 03:23:34.383411 47049167324416 torch/_dynamo/guards.py:2611] [4/3] [__recompiles]     - tensor 'L['row']' dispatch key set mismatch. expected DispatchKeySet(CPU, BackendSelect, ADInplaceOrView, AutogradCPU), actual DispatchKeySet(CUDA, BackendSelect, ADInplaceOrView, AutogradCUDA)
V0421 03:23:34.383411 47049167324416 torch/_dynamo/guards.py:2611] [4/3] [__recompiles]     - L['num_dst'] == 3524                                        
V0421 03:23:34.383411 47049167324416 torch/_dynamo/guards.py:2611] [4/3] [__recompiles]     - L['num_dst'] == 3524                                        
V0421 03:23:34.418472 47049167324416 torch/_dynamo/guards.py:2611] [8/3] [__recompiles] Recompiling function torch_dynamo_resume_in_create_unitgraph_from_coo_at_1296 in /work2/09808/mp46753/frontera/ML_Project/miniconda3/envs/txgnn_env/lib/python3.8/site-packages/dgl/heterograph_index.py:1296
V0421 03:23:34.418472 47049167324416 torch/_dynamo/guards.py:2611] [8/3] [__recompiles]     triggered by the following guard failure(s):
V0421 03:23:34.418472 47049167324416 torch/_dynamo/guards.py:2611] [8/3] [__recompiles]     - tensor 'L['col']' dispatch key set mismatch. expected DispatchKeySet(CPU, BackendSelect, ADInplaceOrView, AutogradCPU), actual DispatchKeySet(CUDA, BackendSelect, ADInplaceOrView, AutogradCUDA)
V0421 03:23:34.418472 47049167324416 torch/_dynamo/guards.py:2611] [8/3] [__recompiles]     - L['___stack2'] == 3524                                      
V0421 03:23:34.418472 47049167324416 torch/_dynamo/guards.py:2611] [8/3] [__recompiles]     - L['___stack2'] == 3524                                      
V0421 03:23:34.434189 47049167324416 torch/_dynamo/guards.py:2611] [15/1] [__recompiles] Recompiling function _dispatch in /work2/09808/mp46753/frontera/ML_Project/miniconda3/envs/txgnn_env/lib/python3.8/site-packages/dgl/utils/internal.py:65
V0421 03:23:34.434189 47049167324416 torch/_dynamo/guards.py:2611] [15/1] [__recompiles]     triggered by the following guard failure(s):
V0421 03:23:34.434189 47049167324416 torch/_dynamo/guards.py:2611] [15/1] [__recompiles]     - L['data'][0] == 3524                                        
V0421 03:23:34.504787 47049167324416 torch/_dynamo/guards.py:2611] [29/1] [__recompiles] Recompiling function torch_dynamo_resume_in_add_edges_at_572 in /work2/09808/mp46753/frontera/ML_Project/miniconda3/envs/txgnn_env/lib/python3.8/site-packages/dgl/heterograph.py:572
V0421 03:23:34.504787 47049167324416 torch/_dynamo/guards.py:2611] [29/1] [__recompiles]     triggered by the following guard failure(s):
V0421 03:23:34.504787 47049167324416 torch/_dynamo/guards.py:2611] [29/1] [__recompiles]     - L['___stack0'] == 3524                                      
V0421 03:23:34.522060 47049167324416 torch/_dynamo/guards.py:2611] [30/2] [__recompiles] Recompiling function torch_dynamo_resume_in_add_edges_at_573 in /work2/09808/mp46753/frontera/ML_Project/miniconda3/envs/txgnn_env/lib/python3.8/site-packages/dgl/heterograph.py:573
V0421 03:23:34.522060 47049167324416 torch/_dynamo/guards.py:2611] [30/2] [__recompiles]     triggered by the following guard failure(s):
V0421 03:23:34.522060 47049167324416 torch/_dynamo/guards.py:2611] [30/2] [__recompiles]     - L['___stack0'] == 3524                                      
V0421 03:23:34.522060 47049167324416 torch/_dynamo/guards.py:2611] [30/2] [__recompiles]     - tensor 'L['u']' size mismatch at index 0. expected 4952, actual 4861
V0421 03:23:34.781430 47049167324416 torch/_dynamo/guards.py:2611] [34/3] [__recompiles] Recompiling function cat in /work2/09808/mp46753/frontera/ML_Project/miniconda3/envs/txgnn_env/lib/python3.8/site-packages/dgl/backend/pytorch/tensor.py:222
V0421 03:23:34.781430 47049167324416 torch/_dynamo/guards.py:2611] [34/3] [__recompiles]     triggered by the following guard failure(s):
V0421 03:23:34.781430 47049167324416 torch/_dynamo/guards.py:2611] [34/3] [__recompiles]     - 2 <= L['seq'][0].size()[0] <= 2147483645                      # _dynamo/output_graph.py:452 in init_ambient_guards
V0421 03:23:34.781430 47049167324416 torch/_dynamo/guards.py:2611] [34/3] [__recompiles]     - tensor 'L['seq'][1]' size mismatch at index 0. expected 4952, actual 4861
V0421 03:23:34.781430 47049167324416 torch/_dynamo/guards.py:2611] [34/3] [__recompiles]     - tensor 'L['seq'][1]' size mismatch at index 0. expected 4952, actual 4861
V0421 03:23:35.018162 47049167324416 torch/_dynamo/guards.py:2611] [4/4] [__recompiles] Recompiling function create_unitgraph_from_coo in /work2/09808/mp46753/frontera/ML_Project/miniconda3/envs/txgnn_env/lib/python3.8/site-packages/dgl/heterograph_index.py:1254
V0421 03:23:35.018162 47049167324416 torch/_dynamo/guards.py:2611] [4/4] [__recompiles]     triggered by the following guard failure(s):
V0421 03:23:35.018162 47049167324416 torch/_dynamo/guards.py:2611] [4/4] [__recompiles]     - tensor 'L['row']' size mismatch at index 0. expected 0, actual 4861
V0421 03:23:35.018162 47049167324416 torch/_dynamo/guards.py:2611] [4/4] [__recompiles]     - tensor 'L['row']' dispatch key set mismatch. expected DispatchKeySet(CPU, BackendSelect, ADInplaceOrView, AutogradCPU), actual DispatchKeySet(CUDA, BackendSelect, ADInplaceOrView, AutogradCUDA)
V0421 03:23:35.018162 47049167324416 torch/_dynamo/guards.py:2611] [4/4] [__recompiles]     - L['num_dst'] == 3524                                        
V0421 03:23:35.018162 47049167324416 torch/_dynamo/guards.py:2611] [4/4] [__recompiles]     - tensor 'L['row']' size mismatch at index 0. expected 0, actual 4861
V0421 03:23:35.056335 47049167324416 torch/_dynamo/guards.py:2611] [8/4] [__recompiles] Recompiling function torch_dynamo_resume_in_create_unitgraph_from_coo_at_1296 in /work2/09808/mp46753/frontera/ML_Project/miniconda3/envs/txgnn_env/lib/python3.8/site-packages/dgl/heterograph_index.py:1296
V0421 03:23:35.056335 47049167324416 torch/_dynamo/guards.py:2611] [8/4] [__recompiles]     triggered by the following guard failure(s):
V0421 03:23:35.056335 47049167324416 torch/_dynamo/guards.py:2611] [8/4] [__recompiles]     - tensor 'L['col']' size mismatch at index 0. expected 0, actual 4861
V0421 03:23:35.056335 47049167324416 torch/_dynamo/guards.py:2611] [8/4] [__recompiles]     - tensor 'L['col']' dispatch key set mismatch. expected DispatchKeySet(CPU, BackendSelect, ADInplaceOrView, AutogradCPU), actual DispatchKeySet(CUDA, BackendSelect, ADInplaceOrView, AutogradCUDA)
V0421 03:23:35.056335 47049167324416 torch/_dynamo/guards.py:2611] [8/4] [__recompiles]     - L['___stack2'] == 3524                                      
V0421 03:23:35.056335 47049167324416 torch/_dynamo/guards.py:2611] [8/4] [__recompiles]     - tensor 'L['col']' size mismatch at index 0. expected 0, actual 4861
V0421 03:23:35.079153 47049167324416 torch/_dynamo/guards.py:2611] [41/1] [__recompiles] Recompiling function spmatrix in /work2/09808/mp46753/frontera/ML_Project/miniconda3/envs/txgnn_env/lib/python3.8/site-packages/dgl/sparse/sparse_matrix.py:763
V0421 03:23:35.079153 47049167324416 torch/_dynamo/guards.py:2611] [41/1] [__recompiles]     triggered by the following guard failure(s):
V0421 03:23:35.079153 47049167324416 torch/_dynamo/guards.py:2611] [41/1] [__recompiles]     - tensor 'L['indices']' stride mismatch at index 0. expected 11872, actual 11690
V0421 03:23:35.338348 47049167324416 torch/_dynamo/guards.py:2611] [47/1] [__recompiles] Recompiling function arange in /work2/09808/mp46753/frontera/ML_Project/miniconda3/envs/txgnn_env/lib/python3.8/site-packages/dgl/backend/pytorch/tensor.py:406
V0421 03:23:35.338348 47049167324416 torch/_dynamo/guards.py:2611] [47/1] [__recompiles]     triggered by the following guard failure(s):
V0421 03:23:35.338348 47049167324416 torch/_dynamo/guards.py:2611] [47/1] [__recompiles]     - L['stop'] == 3524                                           
V0421 03:23:35.558903 47049167324416 torch/_dynamo/guards.py:2611] [48/1] [__recompiles] Recompiling function in_degrees in /work2/09808/mp46753/frontera/ML_Project/miniconda3/envs/txgnn_env/lib/python3.8/site-packages/dgl/heterograph_index.py:702
V0421 03:23:35.558903 47049167324416 torch/_dynamo/guards.py:2611] [48/1] [__recompiles]     triggered by the following guard failure(s):
V0421 03:23:35.558903 47049167324416 torch/_dynamo/guards.py:2611] [48/1] [__recompiles]     - tensor 'L['v']' size mismatch at index 0. expected 3524, actual 3433
V0421 03:23:35.577521 47049167324416 torch/_dynamo/guards.py:2611] [50/1] [__recompiles] Recompiling function has_nodes in /work2/09808/mp46753/frontera/ML_Project/miniconda3/envs/txgnn_env/lib/python3.8/site-packages/dgl/heterograph.py:2878
V0421 03:23:35.577521 47049167324416 torch/_dynamo/guards.py:2611] [50/1] [__recompiles]     triggered by the following guard failure(s):
V0421 03:23:35.577521 47049167324416 torch/_dynamo/guards.py:2611] [50/1] [__recompiles]     - tensor 'L['vid']' size mismatch at index 0. expected 3524, actual 3433
V0421 03:23:35.593367 47049167324416 torch/_dynamo/guards.py:2611] [51/1] [__recompiles] Recompiling function torch_dynamo_resume_in_has_nodes_at_2925 in /work2/09808/mp46753/frontera/ML_Project/miniconda3/envs/txgnn_env/lib/python3.8/site-packages/dgl/heterograph.py:2925
V0421 03:23:35.593367 47049167324416 torch/_dynamo/guards.py:2611] [51/1] [__recompiles]     triggered by the following guard failure(s):
V0421 03:23:35.593367 47049167324416 torch/_dynamo/guards.py:2611] [51/1] [__recompiles]     - tensor 'L['___stack0']' size mismatch at index 0. expected 3524, actual 3433
V0421 03:23:35.826454 47049167324416 torch/_dynamo/guards.py:2611] [52/1] [__recompiles] Recompiling function torch_dynamo_resume_in_has_nodes_at_2926 in /work2/09808/mp46753/frontera/ML_Project/miniconda3/envs/txgnn_env/lib/python3.8/site-packages/dgl/heterograph.py:2926
V0421 03:23:35.826454 47049167324416 torch/_dynamo/guards.py:2611] [52/1] [__recompiles]     triggered by the following guard failure(s):
V0421 03:23:35.826454 47049167324416 torch/_dynamo/guards.py:2611] [52/1] [__recompiles]     - tensor 'L['vid_tensor']' size mismatch at index 0. expected 3524, actual 3433
V0421 03:23:35.849333 47049167324416 torch/_dynamo/guards.py:2611] [53/1] [__recompiles] Recompiling function has_nodes in /work2/09808/mp46753/frontera/ML_Project/miniconda3/envs/txgnn_env/lib/python3.8/site-packages/dgl/heterograph_index.py:444
V0421 03:23:35.849333 47049167324416 torch/_dynamo/guards.py:2611] [53/1] [__recompiles]     triggered by the following guard failure(s):
V0421 03:23:35.849333 47049167324416 torch/_dynamo/guards.py:2611] [53/1] [__recompiles]     - tensor 'L['vids']' size mismatch at index 0. expected 3524, actual 3433
V0421 03:23:35.866923 47049167324416 torch/_dynamo/guards.py:2611] [54/1] [__recompiles] Recompiling function torch_dynamo_resume_in_has_nodes_at_2930 in /work2/09808/mp46753/frontera/ML_Project/miniconda3/envs/txgnn_env/lib/python3.8/site-packages/dgl/heterograph.py:2930
V0421 03:23:35.866923 47049167324416 torch/_dynamo/guards.py:2611] [54/1] [__recompiles]     triggered by the following guard failure(s):
V0421 03:23:35.866923 47049167324416 torch/_dynamo/guards.py:2611] [54/1] [__recompiles]     - tensor 'L['___stack0']' size mismatch at index 0. expected 3524, actual 3433
V0421 03:23:36.088166 47049167324416 torch/_dynamo/guards.py:2611] [55/1] [__recompiles] Recompiling function sum in /work2/09808/mp46753/frontera/ML_Project/miniconda3/envs/txgnn_env/lib/python3.8/site-packages/dgl/backend/pytorch/tensor.py:152
V0421 03:23:36.088166 47049167324416 torch/_dynamo/guards.py:2611] [55/1] [__recompiles]     triggered by the following guard failure(s):
V0421 03:23:36.088166 47049167324416 torch/_dynamo/guards.py:2611] [55/1] [__recompiles]     - tensor 'L['input']' size mismatch at index 0. expected 3524, actual 3433
V0421 03:23:36.167235 47049167324416 torch/_dynamo/guards.py:2611] [57/2] [__recompiles] Recompiling function __setitem__ in /work2/09808/mp46753/frontera/ML_Project/miniconda3/envs/txgnn_env/lib/python3.8/site-packages/dgl/view.py:82
V0421 03:23:36.167235 47049167324416 torch/_dynamo/guards.py:2611] [57/2] [__recompiles]     triggered by the following guard failure(s):
V0421 03:23:36.167235 47049167324416 torch/_dynamo/guards.py:2611] [57/2] [__recompiles]     - tensor 'L['val']' requires_grad mismatch. expected requires_grad=1
V0421 03:23:36.167235 47049167324416 torch/_dynamo/guards.py:2611] [57/2] [__recompiles]     - tensor 'L['val']' size mismatch at index 0. expected 3524, actual 3433
V0421 03:23:36.185052 47049167324416 torch/_dynamo/guards.py:2611] [61/2] [__recompiles] Recompiling function __setitem__ in /work2/09808/mp46753/frontera/ML_Project/miniconda3/envs/txgnn_env/lib/python3.8/site-packages/dgl/frame.py:690
V0421 03:23:36.185052 47049167324416 torch/_dynamo/guards.py:2611] [61/2] [__recompiles]     triggered by the following guard failure(s):
V0421 03:23:36.185052 47049167324416 torch/_dynamo/guards.py:2611] [61/2] [__recompiles]     - tensor 'L['data']' requires_grad mismatch. expected requires_grad=1
V0421 03:23:36.185052 47049167324416 torch/_dynamo/guards.py:2611] [61/2] [__recompiles]     - tensor 'L['data']' size mismatch at index 0. expected 3524, actual 3433
V0421 03:23:36.203090 47049167324416 torch/_dynamo/guards.py:2611] [62/2] [__recompiles] Recompiling function update_column in /work2/09808/mp46753/frontera/ML_Project/miniconda3/envs/txgnn_env/lib/python3.8/site-packages/dgl/frame.py:772
V0421 03:23:36.203090 47049167324416 torch/_dynamo/guards.py:2611] [62/2] [__recompiles]     triggered by the following guard failure(s):
V0421 03:23:36.203090 47049167324416 torch/_dynamo/guards.py:2611] [62/2] [__recompiles]     - tensor 'L['data']' requires_grad mismatch. expected requires_grad=1
V0421 03:23:36.203090 47049167324416 torch/_dynamo/guards.py:2611] [62/2] [__recompiles]     - tensor 'L['data']' size mismatch at index 0. expected 3524, actual 3433
V0421 03:23:36.222626 47049167324416 torch/_dynamo/guards.py:2611] [63/2] [__recompiles] Recompiling function create in /work2/09808/mp46753/frontera/ML_Project/miniconda3/envs/txgnn_env/lib/python3.8/site-packages/dgl/frame.py:472
V0421 03:23:36.222626 47049167324416 torch/_dynamo/guards.py:2611] [63/2] [__recompiles]     triggered by the following guard failure(s):
V0421 03:23:36.222626 47049167324416 torch/_dynamo/guards.py:2611] [63/2] [__recompiles]     - tensor 'L['data']' requires_grad mismatch. expected requires_grad=1
V0421 03:23:36.222626 47049167324416 torch/_dynamo/guards.py:2611] [63/2] [__recompiles]     - tensor 'L['data']' size mismatch at index 0. expected 3524, actual 3433
V0421 03:23:36.238966 47049167324416 torch/_dynamo/guards.py:2611] [64/2] [__recompiles] Recompiling function __init__ in /work2/09808/mp46753/frontera/ML_Project/miniconda3/envs/txgnn_env/lib/python3.8/site-packages/dgl/frame.py:210
V0421 03:23:36.238966 47049167324416 torch/_dynamo/guards.py:2611] [64/2] [__recompiles]     triggered by the following guard failure(s):
V0421 03:23:36.238966 47049167324416 torch/_dynamo/guards.py:2611] [64/2] [__recompiles]     - tensor 'L['storage']' requires_grad mismatch. expected requires_grad=1
V0421 03:23:36.238966 47049167324416 torch/_dynamo/guards.py:2611] [64/2] [__recompiles]     - tensor 'L['storage']' size mismatch at index 0. expected 3524, actual 3433
V0421 03:23:36.257430 47049167324416 torch/_dynamo/guards.py:2611] [65/2] [__recompiles] Recompiling function infer_scheme in /work2/09808/mp46753/frontera/ML_Project/miniconda3/envs/txgnn_env/lib/python3.8/site-packages/dgl/frame.py:151
V0421 03:23:36.257430 47049167324416 torch/_dynamo/guards.py:2611] [65/2] [__recompiles]     triggered by the following guard failure(s):
V0421 03:23:36.257430 47049167324416 torch/_dynamo/guards.py:2611] [65/2] [__recompiles]     - tensor 'L['tensor']' requires_grad mismatch. expected requires_grad=1
V0421 03:23:36.257430 47049167324416 torch/_dynamo/guards.py:2611] [65/2] [__recompiles]     - tensor 'L['tensor']' size mismatch at index 0. expected 3524, actual 3433
V0421 03:23:36.300593 47049167324416 torch/_dynamo/guards.py:2611] [68/2] [__recompiles] Recompiling function func in /work2/09808/mp46753/frontera/ML_Project/miniconda3/envs/txgnn_env/lib/python3.8/site-packages/dgl/ops/spmm.py:213
V0421 03:23:36.300593 47049167324416 torch/_dynamo/guards.py:2611] [68/2] [__recompiles]     triggered by the following guard failure(s):
V0421 03:23:36.300593 47049167324416 torch/_dynamo/guards.py:2611] [68/2] [__recompiles]     - tensor 'L['x']' requires_grad mismatch. expected requires_grad=1
V0421 03:23:36.300593 47049167324416 torch/_dynamo/guards.py:2611] [68/2] [__recompiles]     - tensor 'L['x']' size mismatch at index 0. expected 3524, actual 3433
V0421 03:23:36.314021 47049167324416 torch/_dynamo/guards.py:2611] [70/2] [__recompiles] Recompiling function torch_dynamo_resume_in_gspmm_at_75 in /work2/09808/mp46753/frontera/ML_Project/miniconda3/envs/txgnn_env/lib/python3.8/site-packages/dgl/ops/spmm.py:75
V0421 03:23:36.314021 47049167324416 torch/_dynamo/guards.py:2611] [70/2] [__recompiles]     triggered by the following guard failure(s):
V0421 03:23:36.314021 47049167324416 torch/_dynamo/guards.py:2611] [70/2] [__recompiles]     - tensor 'L['lhs_data']' requires_grad mismatch. expected requires_grad=1
V0421 03:23:36.314021 47049167324416 torch/_dynamo/guards.py:2611] [70/2] [__recompiles]     - tensor 'L['lhs_data']' size mismatch at index 0. expected 3524, actual 3433
V0421 03:23:36.331796 47049167324416 torch/_dynamo/guards.py:2611] [74/2] [__recompiles] Recompiling function forward in /work2/09808/mp46753/frontera/ML_Project/miniconda3/envs/txgnn_env/lib/python3.8/site-packages/dgl/backend/pytorch/sparse.py:163
V0421 03:23:36.331796 47049167324416 torch/_dynamo/guards.py:2611] [74/2] [__recompiles]     triggered by the following guard failure(s):
V0421 03:23:36.331796 47049167324416 torch/_dynamo/guards.py:2611] [74/2] [__recompiles]     - tensor 'L['X']' requires_grad mismatch. expected requires_grad=1
V0421 03:23:36.331796 47049167324416 torch/_dynamo/guards.py:2611] [74/2] [__recompiles]     - tensor 'L['X']' size mismatch at index 0. expected 3524, actual 3433
V0421 03:23:36.347696 47049167324416 torch/_dynamo/guards.py:2611] [77/2] [__recompiles] Recompiling function zeros in /work2/09808/mp46753/frontera/ML_Project/miniconda3/envs/txgnn_env/lib/python3.8/site-packages/dgl/backend/pytorch/tensor.py:287
V0421 03:23:36.347696 47049167324416 torch/_dynamo/guards.py:2611] [77/2] [__recompiles]     triggered by the following guard failure(s):
V0421 03:23:36.347696 47049167324416 torch/_dynamo/guards.py:2611] [77/2] [__recompiles]     - L['shape'][0] == 3524                                       
V0421 03:23:36.347696 47049167324416 torch/_dynamo/guards.py:2611] [77/2] [__recompiles]     - L['shape'][0] == 3524                                       
V0421 03:23:36.518768 47049167324416 torch/_dynamo/guards.py:2611] [79/2] [__recompiles] Recompiling function zerocopy_to_dgl_ndarray_for_write in /work2/09808/mp46753/frontera/ML_Project/miniconda3/envs/txgnn_env/lib/python3.8/site-packages/dgl/backend/pytorch/tensor.py:454
V0421 03:23:36.518768 47049167324416 torch/_dynamo/guards.py:2611] [79/2] [__recompiles]     triggered by the following guard failure(s):
V0421 03:23:36.518768 47049167324416 torch/_dynamo/guards.py:2611] [79/2] [__recompiles]     - tensor 'L['input']' size mismatch at index 0. expected 3524, actual 3433
V0421 03:23:36.518768 47049167324416 torch/_dynamo/guards.py:2611] [79/2] [__recompiles]     - tensor 'L['input']' size mismatch at index 0. expected 3524, actual 3433
V0421 03:23:36.559958 47049167324416 torch/_dynamo/guards.py:2611] [80/2] [__recompiles] Recompiling function check_is_view in /work2/09808/mp46753/frontera/ML_Project/miniconda3/envs/txgnn_env/lib/python3.8/site-packages/dgl/backend/pytorch/tensor.py:441
V0421 03:23:36.559958 47049167324416 torch/_dynamo/guards.py:2611] [80/2] [__recompiles]     triggered by the following guard failure(s):
V0421 03:23:36.559958 47049167324416 torch/_dynamo/guards.py:2611] [80/2] [__recompiles]     - tensor 'L['input']' size mismatch at index 0. expected 3524, actual 3433
V0421 03:23:36.559958 47049167324416 torch/_dynamo/guards.py:2611] [80/2] [__recompiles]     - tensor 'L['input']' size mismatch at index 0. expected 3524, actual 3433
V0421 03:23:36.571393 47049167324416 torch/_dynamo/guards.py:2611] [81/2] [__recompiles] Recompiling function torch_dynamo_resume_in_check_is_view_at_443 in /work2/09808/mp46753/frontera/ML_Project/miniconda3/envs/txgnn_env/lib/python3.8/site-packages/dgl/backend/pytorch/tensor.py:443
V0421 03:23:36.571393 47049167324416 torch/_dynamo/guards.py:2611] [81/2] [__recompiles]     triggered by the following guard failure(s):
V0421 03:23:36.571393 47049167324416 torch/_dynamo/guards.py:2611] [81/2] [__recompiles]     - tensor 'L['input']' size mismatch at index 0. expected 3524, actual 3433
V0421 03:23:36.571393 47049167324416 torch/_dynamo/guards.py:2611] [81/2] [__recompiles]     - tensor 'L['input']' size mismatch at index 0. expected 3524, actual 3433
V0421 03:23:36.583948 47049167324416 torch/_dynamo/guards.py:2611] [82/2] [__recompiles] Recompiling function torch_dynamo_resume_in_zerocopy_to_dgl_ndarray_for_write_at_461 in /work2/09808/mp46753/frontera/ML_Project/miniconda3/envs/txgnn_env/lib/python3.8/site-packages/dgl/backend/pytorch/tensor.py:461
V0421 03:23:36.583948 47049167324416 torch/_dynamo/guards.py:2611] [82/2] [__recompiles]     triggered by the following guard failure(s):
V0421 03:23:36.583948 47049167324416 torch/_dynamo/guards.py:2611] [82/2] [__recompiles]     - tensor 'L['input']' size mismatch at index 0. expected 3524, actual 3433
V0421 03:23:36.583948 47049167324416 torch/_dynamo/guards.py:2611] [82/2] [__recompiles]     - tensor 'L['input']' size mismatch at index 0. expected 3524, actual 3433
V0421 03:23:36.601305 47049167324416 torch/_dynamo/guards.py:2611] [88/1] [__recompiles] Recompiling function forward in /work2/09808/mp46753/frontera/ML_Project/VQGraph/vq.py:569
V0421 03:23:36.601305 47049167324416 torch/_dynamo/guards.py:2611] [88/1] [__recompiles]     triggered by the following guard failure(s):
V0421 03:23:36.601305 47049167324416 torch/_dynamo/guards.py:2611] [88/1] [__recompiles]     - tensor 'L['x']' size mismatch at index 0. expected 3524, actual 3433
V0421 03:23:36.691021 47049167324416 torch/_dynamo/guards.py:2611] [89/1] [__recompiles] Recompiling function torch_dynamo_resume_in_forward_at_590 in /work2/09808/mp46753/frontera/ML_Project/VQGraph/vq.py:590
V0421 03:23:36.691021 47049167324416 torch/_dynamo/guards.py:2611] [89/1] [__recompiles]     triggered by the following guard failure(s):
V0421 03:23:36.691021 47049167324416 torch/_dynamo/guards.py:2611] [89/1] [__recompiles]     - tensor 'L['___stack0']' size mismatch at index 0. expected 3524, actual 3433
V0421 03:23:36.752747 47049167324416 torch/_dynamo/guards.py:2611] [90/1] [__recompiles] Recompiling function forward in /work2/09808/mp46753/frontera/ML_Project/VQGraph/vq.py:418
V0421 03:23:36.752747 47049167324416 torch/_dynamo/guards.py:2611] [90/1] [__recompiles]     triggered by the following guard failure(s):
V0421 03:23:36.752747 47049167324416 torch/_dynamo/guards.py:2611] [90/1] [__recompiles]     - tensor 'L['x']' size mismatch at index 0. expected 3524, actual 3433
V0421 03:23:37.343969 47049167324416 torch/_dynamo/guards.py:2611] [93/1] [__recompiles] Recompiling function torch_dynamo_resume_in_forward_at_432 in /work2/09808/mp46753/frontera/ML_Project/VQGraph/vq.py:432
V0421 03:23:37.343969 47049167324416 torch/_dynamo/guards.py:2611] [93/1] [__recompiles]     triggered by the following guard failure(s):
V0421 03:23:37.343969 47049167324416 torch/_dynamo/guards.py:2611] [93/1] [__recompiles]     - L['shape'][1] == 3524                                       
V0421 03:23:38.621783 47049167324416 torch/_dynamo/guards.py:2611] [94/1] [__recompiles] Recompiling function torch_dynamo_resume_in_forward_at_595 in /work2/09808/mp46753/frontera/ML_Project/VQGraph/vq.py:595
V0421 03:23:38.621783 47049167324416 torch/_dynamo/guards.py:2611] [94/1] [__recompiles]     triggered by the following guard failure(s):
V0421 03:23:38.621783 47049167324416 torch/_dynamo/guards.py:2611] [94/1] [__recompiles]     - tensor 'L['x']' size mismatch at index 0. expected 3524, actual 3433
V0421 03:23:38.879034 47049167324416 torch/_dynamo/guards.py:2611] [95/1] [__recompiles] Recompiling function torch_dynamo_resume_in_forward_at_604 in /work2/09808/mp46753/frontera/ML_Project/VQGraph/vq.py:604
V0421 03:23:38.879034 47049167324416 torch/_dynamo/guards.py:2611] [95/1] [__recompiles]     triggered by the following guard failure(s):
V0421 03:23:38.879034 47049167324416 torch/_dynamo/guards.py:2611] [95/1] [__recompiles]     - tensor 'L['x']' size mismatch at index 0. expected 3524, actual 3433
V0421 03:23:39.600211 47049167324416 torch/_dynamo/guards.py:2611] [96/1] [__recompiles] Recompiling function torch_dynamo_resume_in_forward_at_648 in /work2/09808/mp46753/frontera/ML_Project/VQGraph/vq.py:648
V0421 03:23:39.600211 47049167324416 torch/_dynamo/guards.py:2611] [96/1] [__recompiles]     triggered by the following guard failure(s):
V0421 03:23:39.600211 47049167324416 torch/_dynamo/guards.py:2611] [96/1] [__recompiles]     - tensor 'L['___stack0']' size mismatch at index 0. expected 3524, actual 3433
V0421 03:23:39.704962 47049167324416 torch/_dynamo/guards.py:2611] [57/3] [__recompiles] Recompiling function __setitem__ in /work2/09808/mp46753/frontera/ML_Project/miniconda3/envs/txgnn_env/lib/python3.8/site-packages/dgl/view.py:82
V0421 03:23:39.704962 47049167324416 torch/_dynamo/guards.py:2611] [57/3] [__recompiles]     triggered by the following guard failure(s):
V0421 03:23:39.704962 47049167324416 torch/_dynamo/guards.py:2611] [57/3] [__recompiles]     - tensor 'L['val']' requires_grad mismatch. expected requires_grad=0
V0421 03:23:39.704962 47049167324416 torch/_dynamo/guards.py:2611] [57/3] [__recompiles]     - tensor 'L['val']' size mismatch at index 0. expected 3524, actual 3433
V0421 03:23:39.704962 47049167324416 torch/_dynamo/guards.py:2611] [57/3] [__recompiles]     - tensor 'L['val']' requires_grad mismatch. expected requires_grad=0
V0421 03:23:39.725316 47049167324416 torch/_dynamo/guards.py:2611] [61/3] [__recompiles] Recompiling function __setitem__ in /work2/09808/mp46753/frontera/ML_Project/miniconda3/envs/txgnn_env/lib/python3.8/site-packages/dgl/frame.py:690
V0421 03:23:39.725316 47049167324416 torch/_dynamo/guards.py:2611] [61/3] [__recompiles]     triggered by the following guard failure(s):
V0421 03:23:39.725316 47049167324416 torch/_dynamo/guards.py:2611] [61/3] [__recompiles]     - tensor 'L['data']' requires_grad mismatch. expected requires_grad=0
V0421 03:23:39.725316 47049167324416 torch/_dynamo/guards.py:2611] [61/3] [__recompiles]     - tensor 'L['data']' size mismatch at index 0. expected 3524, actual 3433
V0421 03:23:39.725316 47049167324416 torch/_dynamo/guards.py:2611] [61/3] [__recompiles]     - tensor 'L['data']' requires_grad mismatch. expected requires_grad=0
V0421 03:23:39.745742 47049167324416 torch/_dynamo/guards.py:2611] [62/3] [__recompiles] Recompiling function update_column in /work2/09808/mp46753/frontera/ML_Project/miniconda3/envs/txgnn_env/lib/python3.8/site-packages/dgl/frame.py:772
V0421 03:23:39.745742 47049167324416 torch/_dynamo/guards.py:2611] [62/3] [__recompiles]     triggered by the following guard failure(s):
V0421 03:23:39.745742 47049167324416 torch/_dynamo/guards.py:2611] [62/3] [__recompiles]     - tensor 'L['data']' requires_grad mismatch. expected requires_grad=0
V0421 03:23:39.745742 47049167324416 torch/_dynamo/guards.py:2611] [62/3] [__recompiles]     - tensor 'L['data']' size mismatch at index 0. expected 3524, actual 3433
V0421 03:23:39.745742 47049167324416 torch/_dynamo/guards.py:2611] [62/3] [__recompiles]     - tensor 'L['data']' requires_grad mismatch. expected requires_grad=0
V0421 03:23:39.766593 47049167324416 torch/_dynamo/guards.py:2611] [63/3] [__recompiles] Recompiling function create in /work2/09808/mp46753/frontera/ML_Project/miniconda3/envs/txgnn_env/lib/python3.8/site-packages/dgl/frame.py:472
V0421 03:23:39.766593 47049167324416 torch/_dynamo/guards.py:2611] [63/3] [__recompiles]     triggered by the following guard failure(s):
V0421 03:23:39.766593 47049167324416 torch/_dynamo/guards.py:2611] [63/3] [__recompiles]     - tensor 'L['data']' requires_grad mismatch. expected requires_grad=0
V0421 03:23:39.766593 47049167324416 torch/_dynamo/guards.py:2611] [63/3] [__recompiles]     - tensor 'L['data']' size mismatch at index 0. expected 3524, actual 3433
V0421 03:23:39.766593 47049167324416 torch/_dynamo/guards.py:2611] [63/3] [__recompiles]     - tensor 'L['data']' requires_grad mismatch. expected requires_grad=0
V0421 03:23:39.785310 47049167324416 torch/_dynamo/guards.py:2611] [64/3] [__recompiles] Recompiling function __init__ in /work2/09808/mp46753/frontera/ML_Project/miniconda3/envs/txgnn_env/lib/python3.8/site-packages/dgl/frame.py:210
V0421 03:23:39.785310 47049167324416 torch/_dynamo/guards.py:2611] [64/3] [__recompiles]     triggered by the following guard failure(s):
V0421 03:23:39.785310 47049167324416 torch/_dynamo/guards.py:2611] [64/3] [__recompiles]     - tensor 'L['storage']' requires_grad mismatch. expected requires_grad=0
V0421 03:23:39.785310 47049167324416 torch/_dynamo/guards.py:2611] [64/3] [__recompiles]     - tensor 'L['storage']' size mismatch at index 0. expected 3524, actual 3433
V0421 03:23:39.785310 47049167324416 torch/_dynamo/guards.py:2611] [64/3] [__recompiles]     - tensor 'L['storage']' requires_grad mismatch. expected requires_grad=0
V0421 03:23:39.806129 47049167324416 torch/_dynamo/guards.py:2611] [65/3] [__recompiles] Recompiling function infer_scheme in /work2/09808/mp46753/frontera/ML_Project/miniconda3/envs/txgnn_env/lib/python3.8/site-packages/dgl/frame.py:151
V0421 03:23:39.806129 47049167324416 torch/_dynamo/guards.py:2611] [65/3] [__recompiles]     triggered by the following guard failure(s):
V0421 03:23:39.806129 47049167324416 torch/_dynamo/guards.py:2611] [65/3] [__recompiles]     - tensor 'L['tensor']' requires_grad mismatch. expected requires_grad=0
V0421 03:23:39.806129 47049167324416 torch/_dynamo/guards.py:2611] [65/3] [__recompiles]     - tensor 'L['tensor']' size mismatch at index 0. expected 3524, actual 3433
V0421 03:23:39.806129 47049167324416 torch/_dynamo/guards.py:2611] [65/3] [__recompiles]     - tensor 'L['tensor']' requires_grad mismatch. expected requires_grad=0
V0421 03:23:39.854436 47049167324416 torch/_dynamo/guards.py:2611] [68/3] [__recompiles] Recompiling function func in /work2/09808/mp46753/frontera/ML_Project/miniconda3/envs/txgnn_env/lib/python3.8/site-packages/dgl/ops/spmm.py:213
V0421 03:23:39.854436 47049167324416 torch/_dynamo/guards.py:2611] [68/3] [__recompiles]     triggered by the following guard failure(s):
V0421 03:23:39.854436 47049167324416 torch/_dynamo/guards.py:2611] [68/3] [__recompiles]     - tensor 'L['x']' requires_grad mismatch. expected requires_grad=0
V0421 03:23:39.854436 47049167324416 torch/_dynamo/guards.py:2611] [68/3] [__recompiles]     - tensor 'L['x']' size mismatch at index 0. expected 3524, actual 3433
V0421 03:23:39.854436 47049167324416 torch/_dynamo/guards.py:2611] [68/3] [__recompiles]     - tensor 'L['x']' requires_grad mismatch. expected requires_grad=0
V0421 03:23:39.868981 47049167324416 torch/_dynamo/guards.py:2611] [70/3] [__recompiles] Recompiling function torch_dynamo_resume_in_gspmm_at_75 in /work2/09808/mp46753/frontera/ML_Project/miniconda3/envs/txgnn_env/lib/python3.8/site-packages/dgl/ops/spmm.py:75
V0421 03:23:39.868981 47049167324416 torch/_dynamo/guards.py:2611] [70/3] [__recompiles]     triggered by the following guard failure(s):
V0421 03:23:39.868981 47049167324416 torch/_dynamo/guards.py:2611] [70/3] [__recompiles]     - tensor 'L['lhs_data']' requires_grad mismatch. expected requires_grad=0
V0421 03:23:39.868981 47049167324416 torch/_dynamo/guards.py:2611] [70/3] [__recompiles]     - tensor 'L['lhs_data']' size mismatch at index 0. expected 3524, actual 3433
V0421 03:23:39.868981 47049167324416 torch/_dynamo/guards.py:2611] [70/3] [__recompiles]     - tensor 'L['lhs_data']' requires_grad mismatch. expected requires_grad=0
V0421 03:23:39.890290 47049167324416 torch/_dynamo/guards.py:2611] [74/3] [__recompiles] Recompiling function forward in /work2/09808/mp46753/frontera/ML_Project/miniconda3/envs/txgnn_env/lib/python3.8/site-packages/dgl/backend/pytorch/sparse.py:163
V0421 03:23:39.890290 47049167324416 torch/_dynamo/guards.py:2611] [74/3] [__recompiles]     triggered by the following guard failure(s):
V0421 03:23:39.890290 47049167324416 torch/_dynamo/guards.py:2611] [74/3] [__recompiles]     - tensor 'L['X']' requires_grad mismatch. expected requires_grad=0
V0421 03:23:39.890290 47049167324416 torch/_dynamo/guards.py:2611] [74/3] [__recompiles]     - tensor 'L['X']' size mismatch at index 0. expected 3524, actual 3433
V0421 03:23:39.890290 47049167324416 torch/_dynamo/guards.py:2611] [74/3] [__recompiles]     - tensor 'L['X']' requires_grad mismatch. expected requires_grad=0
/work2/09808/mp46753/frontera/ML_Project/miniconda3/envs/txgnn_env/lib/python3.8/site-packages/dgl/dataloading/dataloader.py:1144: DGLWarning: Dataloader CPU affinity opt is not enabled, consider switching it on (see enable_cpu_affinity() or CPU best practices for DGL [https://docs.dgl.ai/tutorials/cpu/cpu_best_practises.html])
  dgl_warning(
/work2/09808/mp46753/frontera/ML_Project/miniconda3/envs/txgnn_env/lib/python3.8/site-packages/dgl/heterograph.py:92: DGLWarning: Recommend creating graphs by `dgl.graph(data)` instead of `dgl.DGLGraph(data)`.
  dgl_warning(
V0421 03:23:40.952989 47049167324416 torch/_dynamo/guards.py:2611] [15/2] [__recompiles] Recompiling function _dispatch in /work2/09808/mp46753/frontera/ML_Project/miniconda3/envs/txgnn_env/lib/python3.8/site-packages/dgl/utils/internal.py:65
V0421 03:23:40.952989 47049167324416 torch/_dynamo/guards.py:2611] [15/2] [__recompiles]     triggered by the following guard failure(s):
V0421 03:23:40.952989 47049167324416 torch/_dynamo/guards.py:2611] [15/2] [__recompiles]     - L['data'][0] == 3433                                          # _dynamo/output_graph.py:452 in init_ambient_guards
V0421 03:23:40.952989 47049167324416 torch/_dynamo/guards.py:2611] [15/2] [__recompiles]     - L['data'][0] == 3524                                        
/work2/09808/mp46753/frontera/ML_Project/miniconda3/envs/txgnn_env/lib/python3.8/site-packages/dgl/dataloading/dataloader.py:1144: DGLWarning: Dataloader CPU affinity opt is not enabled, consider switching it on (see enable_cpu_affinity() or CPU best practices for DGL [https://docs.dgl.ai/tutorials/cpu/cpu_best_practises.html])
  dgl_warning(
/work2/09808/mp46753/frontera/ML_Project/miniconda3/envs/txgnn_env/lib/python3.8/site-packages/dgl/heterograph.py:92: DGLWarning: Recommend creating graphs by `dgl.graph(data)` instead of `dgl.DGLGraph(data)`.
  dgl_warning(
V0421 03:23:45.234143 47049167324416 torch/_dynamo/guards.py:2611] [15/3] [__recompiles] Recompiling function _dispatch in /work2/09808/mp46753/frontera/ML_Project/miniconda3/envs/txgnn_env/lib/python3.8/site-packages/dgl/utils/internal.py:65
V0421 03:23:45.234143 47049167324416 torch/_dynamo/guards.py:2611] [15/3] [__recompiles]     triggered by the following guard failure(s):
V0421 03:23:45.234143 47049167324416 torch/_dynamo/guards.py:2611] [15/3] [__recompiles]     - L['data'][0] == 3474                                          # _dynamo/output_graph.py:452 in init_ambient_guards
V0421 03:23:45.234143 47049167324416 torch/_dynamo/guards.py:2611] [15/3] [__recompiles]     - L['data'][0] == 3433                                          # _dynamo/output_graph.py:452 in init_ambient_guards
V0421 03:23:45.234143 47049167324416 torch/_dynamo/guards.py:2611] [15/3] [__recompiles]     - L['data'][0] == 3524                                        
/work2/09808/mp46753/frontera/ML_Project/miniconda3/envs/txgnn_env/lib/python3.8/site-packages/dgl/dataloading/dataloader.py:1144: DGLWarning: Dataloader CPU affinity opt is not enabled, consider switching it on (see enable_cpu_affinity() or CPU best practices for DGL [https://docs.dgl.ai/tutorials/cpu/cpu_best_practises.html])
  dgl_warning(
/work2/09808/mp46753/frontera/ML_Project/miniconda3/envs/txgnn_env/lib/python3.8/site-packages/dgl/heterograph.py:92: DGLWarning: Recommend creating graphs by `dgl.graph(data)` instead of `dgl.DGLGraph(data)`.
  dgl_warning(
V0421 03:23:49.544331 47049167324416 torch/_dynamo/guards.py:2611] [15/4] [__recompiles] Recompiling function _dispatch in /work2/09808/mp46753/frontera/ML_Project/miniconda3/envs/txgnn_env/lib/python3.8/site-packages/dgl/utils/internal.py:65
V0421 03:23:49.544331 47049167324416 torch/_dynamo/guards.py:2611] [15/4] [__recompiles]     triggered by the following guard failure(s):
V0421 03:23:49.544331 47049167324416 torch/_dynamo/guards.py:2611] [15/4] [__recompiles]     - L['data'][0] == 3432                                          # _dynamo/output_graph.py:452 in init_ambient_guards
V0421 03:23:49.544331 47049167324416 torch/_dynamo/guards.py:2611] [15/4] [__recompiles]     - L['data'][0] == 3474                                          # _dynamo/output_graph.py:452 in init_ambient_guards
V0421 03:23:49.544331 47049167324416 torch/_dynamo/guards.py:2611] [15/4] [__recompiles]     - L['data'][0] == 3433                                          # _dynamo/output_graph.py:452 in init_ambient_guards
V0421 03:23:49.544331 47049167324416 torch/_dynamo/guards.py:2611] [15/4] [__recompiles]     - L['data'][0] == 3524                                        
/work2/09808/mp46753/frontera/ML_Project/miniconda3/envs/txgnn_env/lib/python3.8/site-packages/dgl/dataloading/dataloader.py:1144: DGLWarning: Dataloader CPU affinity opt is not enabled, consider switching it on (see enable_cpu_affinity() or CPU best practices for DGL [https://docs.dgl.ai/tutorials/cpu/cpu_best_practises.html])
  dgl_warning(
/work2/09808/mp46753/frontera/ML_Project/miniconda3/envs/txgnn_env/lib/python3.8/site-packages/dgl/heterograph.py:92: DGLWarning: Recommend creating graphs by `dgl.graph(data)` instead of `dgl.DGLGraph(data)`.
  dgl_warning(
V0421 03:23:53.832751 47049167324416 torch/_dynamo/guards.py:2611] [15/5] [__recompiles] Recompiling function _dispatch in /work2/09808/mp46753/frontera/ML_Project/miniconda3/envs/txgnn_env/lib/python3.8/site-packages/dgl/utils/internal.py:65
V0421 03:23:53.832751 47049167324416 torch/_dynamo/guards.py:2611] [15/5] [__recompiles]     triggered by the following guard failure(s):
V0421 03:23:53.832751 47049167324416 torch/_dynamo/guards.py:2611] [15/5] [__recompiles]     - L['data'][0] == 3502                                          # _dynamo/output_graph.py:452 in init_ambient_guards
V0421 03:23:53.832751 47049167324416 torch/_dynamo/guards.py:2611] [15/5] [__recompiles]     - L['data'][0] == 3432                                          # _dynamo/output_graph.py:452 in init_ambient_guards
V0421 03:23:53.832751 47049167324416 torch/_dynamo/guards.py:2611] [15/5] [__recompiles]     - L['data'][0] == 3474                                          # _dynamo/output_graph.py:452 in init_ambient_guards
V0421 03:23:53.832751 47049167324416 torch/_dynamo/guards.py:2611] [15/5] [__recompiles]     - L['data'][0] == 3433                                          # _dynamo/output_graph.py:452 in init_ambient_guards
V0421 03:23:53.832751 47049167324416 torch/_dynamo/guards.py:2611] [15/5] [__recompiles]     - L['data'][0] == 3524                                        
/work2/09808/mp46753/frontera/ML_Project/miniconda3/envs/txgnn_env/lib/python3.8/site-packages/dgl/dataloading/dataloader.py:1144: DGLWarning: Dataloader CPU affinity opt is not enabled, consider switching it on (see enable_cpu_affinity() or CPU best practices for DGL [https://docs.dgl.ai/tutorials/cpu/cpu_best_practises.html])
  dgl_warning(
/work2/09808/mp46753/frontera/ML_Project/miniconda3/envs/txgnn_env/lib/python3.8/site-packages/dgl/heterograph.py:92: DGLWarning: Recommend creating graphs by `dgl.graph(data)` instead of `dgl.DGLGraph(data)`.
  dgl_warning(
V0421 03:23:58.148612 47049167324416 torch/_dynamo/guards.py:2611] [15/6] [__recompiles] Recompiling function _dispatch in /work2/09808/mp46753/frontera/ML_Project/miniconda3/envs/txgnn_env/lib/python3.8/site-packages/dgl/utils/internal.py:65
V0421 03:23:58.148612 47049167324416 torch/_dynamo/guards.py:2611] [15/6] [__recompiles]     triggered by the following guard failure(s):
V0421 03:23:58.148612 47049167324416 torch/_dynamo/guards.py:2611] [15/6] [__recompiles]     - L['data'][0] == 3473                                          # _dynamo/output_graph.py:452 in init_ambient_guards
V0421 03:23:58.148612 47049167324416 torch/_dynamo/guards.py:2611] [15/6] [__recompiles]     - L['data'][0] == 3502                                          # _dynamo/output_graph.py:452 in init_ambient_guards
V0421 03:23:58.148612 47049167324416 torch/_dynamo/guards.py:2611] [15/6] [__recompiles]     - L['data'][0] == 3432                                          # _dynamo/output_graph.py:452 in init_ambient_guards
V0421 03:23:58.148612 47049167324416 torch/_dynamo/guards.py:2611] [15/6] [__recompiles]     - L['data'][0] == 3474                                          # _dynamo/output_graph.py:452 in init_ambient_guards
V0421 03:23:58.148612 47049167324416 torch/_dynamo/guards.py:2611] [15/6] [__recompiles]     - L['data'][0] == 3433                                          # _dynamo/output_graph.py:452 in init_ambient_guards
V0421 03:23:58.148612 47049167324416 torch/_dynamo/guards.py:2611] [15/6] [__recompiles]     - L['data'][0] == 3524                                        
/work2/09808/mp46753/frontera/ML_Project/miniconda3/envs/txgnn_env/lib/python3.8/site-packages/dgl/dataloading/dataloader.py:1144: DGLWarning: Dataloader CPU affinity opt is not enabled, consider switching it on (see enable_cpu_affinity() or CPU best practices for DGL [https://docs.dgl.ai/tutorials/cpu/cpu_best_practises.html])
  dgl_warning(
/work2/09808/mp46753/frontera/ML_Project/miniconda3/envs/txgnn_env/lib/python3.8/site-packages/dgl/heterograph.py:92: DGLWarning: Recommend creating graphs by `dgl.graph(data)` instead of `dgl.DGLGraph(data)`.
  dgl_warning(
V0421 03:24:02.482920 47049167324416 torch/_dynamo/guards.py:2611] [15/7] [__recompiles] Recompiling function _dispatch in /work2/09808/mp46753/frontera/ML_Project/miniconda3/envs/txgnn_env/lib/python3.8/site-packages/dgl/utils/internal.py:65
V0421 03:24:02.482920 47049167324416 torch/_dynamo/guards.py:2611] [15/7] [__recompiles]     triggered by the following guard failure(s):
V0421 03:24:02.482920 47049167324416 torch/_dynamo/guards.py:2611] [15/7] [__recompiles]     - L['data'][0] == 3526                                          # _dynamo/output_graph.py:452 in init_ambient_guards
V0421 03:24:02.482920 47049167324416 torch/_dynamo/guards.py:2611] [15/7] [__recompiles]     - L['data'][0] == 3473                                          # _dynamo/output_graph.py:452 in init_ambient_guards
V0421 03:24:02.482920 47049167324416 torch/_dynamo/guards.py:2611] [15/7] [__recompiles]     - L['data'][0] == 3502                                          # _dynamo/output_graph.py:452 in init_ambient_guards
V0421 03:24:02.482920 47049167324416 torch/_dynamo/guards.py:2611] [15/7] [__recompiles]     - L['data'][0] == 3432                                          # _dynamo/output_graph.py:452 in init_ambient_guards
V0421 03:24:02.482920 47049167324416 torch/_dynamo/guards.py:2611] [15/7] [__recompiles]     - L['data'][0] == 3474                                          # _dynamo/output_graph.py:452 in init_ambient_guards
V0421 03:24:02.482920 47049167324416 torch/_dynamo/guards.py:2611] [15/7] [__recompiles]     - L['data'][0] == 3433                                          # _dynamo/output_graph.py:452 in init_ambient_guards
V0421 03:24:02.482920 47049167324416 torch/_dynamo/guards.py:2611] [15/7] [__recompiles]     - L['data'][0] == 3524                                        
/work2/09808/mp46753/frontera/ML_Project/miniconda3/envs/txgnn_env/lib/python3.8/site-packages/dgl/dataloading/dataloader.py:1144: DGLWarning: Dataloader CPU affinity opt is not enabled, consider switching it on (see enable_cpu_affinity() or CPU best practices for DGL [https://docs.dgl.ai/tutorials/cpu/cpu_best_practises.html])
  dgl_warning(
/work2/09808/mp46753/frontera/ML_Project/miniconda3/envs/txgnn_env/lib/python3.8/site-packages/dgl/heterograph.py:92: DGLWarning: Recommend creating graphs by `dgl.graph(data)` instead of `dgl.DGLGraph(data)`.
  dgl_warning(
V0421 03:24:06.851594 47049167324416 torch/_dynamo/guards.py:2611] [15/8] [__recompiles] Recompiling function _dispatch in /work2/09808/mp46753/frontera/ML_Project/miniconda3/envs/txgnn_env/lib/python3.8/site-packages/dgl/utils/internal.py:65
V0421 03:24:06.851594 47049167324416 torch/_dynamo/guards.py:2611] [15/8] [__recompiles]     triggered by the following guard failure(s):
V0421 03:24:06.851594 47049167324416 torch/_dynamo/guards.py:2611] [15/8] [__recompiles]     - L['data'][0] == 3499                                          # _dynamo/output_graph.py:452 in init_ambient_guards
V0421 03:24:06.851594 47049167324416 torch/_dynamo/guards.py:2611] [15/8] [__recompiles]     - L['data'][0] == 3526                                          # _dynamo/output_graph.py:452 in init_ambient_guards
V0421 03:24:06.851594 47049167324416 torch/_dynamo/guards.py:2611] [15/8] [__recompiles]     - L['data'][0] == 3473                                          # _dynamo/output_graph.py:452 in init_ambient_guards
V0421 03:24:06.851594 47049167324416 torch/_dynamo/guards.py:2611] [15/8] [__recompiles]     - L['data'][0] == 3502                                          # _dynamo/output_graph.py:452 in init_ambient_guards
V0421 03:24:06.851594 47049167324416 torch/_dynamo/guards.py:2611] [15/8] [__recompiles]     - L['data'][0] == 3432                                          # _dynamo/output_graph.py:452 in init_ambient_guards
V0421 03:24:06.851594 47049167324416 torch/_dynamo/guards.py:2611] [15/8] [__recompiles]     - L['data'][0] == 3474                                          # _dynamo/output_graph.py:452 in init_ambient_guards
V0421 03:24:06.851594 47049167324416 torch/_dynamo/guards.py:2611] [15/8] [__recompiles]     - L['data'][0] == 3433                                          # _dynamo/output_graph.py:452 in init_ambient_guards
V0421 03:24:06.851594 47049167324416 torch/_dynamo/guards.py:2611] [15/8] [__recompiles]     - L['data'][0] == 3524                                        
W0421 03:24:06.851908 47049167324416 torch/_dynamo/convert_frame.py:762] [15/8] torch._dynamo hit config.cache_size_limit (8)
W0421 03:24:06.851908 47049167324416 torch/_dynamo/convert_frame.py:762] [15/8]    function: '_dispatch' (/work2/09808/mp46753/frontera/ML_Project/miniconda3/envs/txgnn_env/lib/python3.8/site-packages/dgl/utils/internal.py:65)
W0421 03:24:06.851908 47049167324416 torch/_dynamo/convert_frame.py:762] [15/8]    last reason: L['data'][0] == 3524                                        
W0421 03:24:06.851908 47049167324416 torch/_dynamo/convert_frame.py:762] [15/8] To log all recompilation reasons, use TORCH_LOGS="recompiles".
W0421 03:24:06.851908 47049167324416 torch/_dynamo/convert_frame.py:762] [15/8] To diagnose recompilation issues, see https://pytorch.org/docs/main/torch.compiler_troubleshooting.html.
/work2/09808/mp46753/frontera/ML_Project/miniconda3/envs/txgnn_env/lib/python3.8/site-packages/dgl/dataloading/dataloader.py:1144: DGLWarning: Dataloader CPU affinity opt is not enabled, consider switching it on (see enable_cpu_affinity() or CPU best practices for DGL [https://docs.dgl.ai/tutorials/cpu/cpu_best_practises.html])
  dgl_warning(
/work2/09808/mp46753/frontera/ML_Project/miniconda3/envs/txgnn_env/lib/python3.8/site-packages/dgl/heterograph.py:92: DGLWarning: Recommend creating graphs by `dgl.graph(data)` instead of `dgl.DGLGraph(data)`.
  dgl_warning(
Compiling model with torch.compile()...
Saved sparsity snapshot to outputs_compiled/tran/a-computer/SAGE/seed_1/sparsity_before.csv
                               layer_name  num_elements   sparsity
0  _orig_mod.encoder.graph_layer_1.weight        588289    0.00017
1    _orig_mod.encoder.graph_layer_1.bias           767  100.00000
2  _orig_mod.encoder.graph_layer_2.weight         98176    0.00000
3    _orig_mod.encoder.graph_layer_2.bias           128  100.00000
4      _orig_mod.encoder.decoder_1.weight        588289    0.00000
5        _orig_mod.encoder.decoder_1.bias           767    0.00000
6      _orig_mod.encoder.decoder_2.weight        588289    0.00000
7        _orig_mod.encoder.decoder_2.bias           767    0.00000
8         _orig_mod.encoder.linear.weight          1280    0.00000
9           _orig_mod.encoder.linear.bias            10    0.00000
Saved sparsity snapshot to outputs_compiled/tran/a-computer/SAGE/seed_1/sparsity_after.csv
                               layer_name  num_elements  sparsity
0  _orig_mod.encoder.graph_layer_1.weight        588289  0.031787
1    _orig_mod.encoder.graph_layer_1.bias           767  0.000000
2  _orig_mod.encoder.graph_layer_2.weight         98176  0.055003
3    _orig_mod.encoder.graph_layer_2.bias           128  0.000000
4      _orig_mod.encoder.decoder_1.weight        588289  0.340309
5        _orig_mod.encoder.decoder_1.bias           767  0.000000
6      _orig_mod.encoder.decoder_2.weight        588289  4.045801
7        _orig_mod.encoder.decoder_2.bias           767  1.825293
8         _orig_mod.encoder.linear.weight          1280  0.000000
9           _orig_mod.encoder.linear.bias            10  0.000000
Saved forward timing report to outputs_compiled/tran/a-computer/SAGE/seed_1/forward_times.csv
                         layer_name  avg_forward_time_ms  num_calls
0         _orig_mod.encoder.dropout             0.000648      14027
1   _orig_mod.encoder.graph_layer_1             0.000901      14027
2   _orig_mod.encoder.graph_layer_2             0.000747      14027
3       _orig_mod.encoder.decoder_1             0.000794      14027
4       _orig_mod.encoder.decoder_2             0.000708      14027
5          _orig_mod.encoder.linear             0.000329      14027
6   _orig_mod.encoder.vq.project_in             0.000628      14027
7  _orig_mod.encoder.vq.project_out             0.019494      14027
8    _orig_mod.encoder.vq._codebook             0.000898      14027
CPU Memory Usage: 1307.21 MB
Max GPU Memory Used: 2081.20 MB
